[2017-11-20 23:36:57,943] olympus.Olympus<Olympus:bb002>:OUTPUT: Parsed config: {'clients': {}, 'replicas': {}, 'failures': {0: {'r1': {'client_request': [{'failure': ('increment_slot', ''), 'trigger_point': {'event_num': 1, 'application': 'pre', 'client_id': 'c0'}}]}}}, 'test_case_name': 'test_increment_slot_t2_multi_client', 't': 2, 'num_client': 2, 'client_timeout': 3000, 'head_timeout': 3000, 'nonhead_timeout': 3000, 'checkpt_interval': 100, 'hosts': {0: 'localhost;'}, 'client_hosts': {'c0': 'localhost;', 'c1': 'localhost;'}, 'replica_hosts': {'r0': '0', 'r1': '0', 'r2': '0', 'r3': '0', 'r4': '0'}, 'workloads': {'c0': ["put('movie','star')", "get('movie')", "append('movie', '123')", "get('movie')", "get('movie');"], 'c1': ["put('movie1','star1')", "get('movie1')", "append('movie1', '123')", "get('movie1')", "get('movie1');"]}}
[2017-11-20 23:36:57,950] olympus.Olympus<Olympus:bb002>:OUTPUT: Generated keys: {'private_keys': {'o1': b'64523384870b9c1081bf60151e908ef8ef08c195bb39c19db1a009baa41d6ba1', 'c0': b'09e0ab577e1adcdc0d6dbab28ed9bd68086700eb38741f715a0aa6c3bd88fda6', 'c1': b'bdcf34d945513f550c3ad755ecf681f1da99b0021a14ff783e06090453ab47c8', 'r0': b'f0cc48cd7a362629549f03e66d698b5be173ab72d4821fa40f776a8b477ab716', 'r1': b'd0264d0372914f2ecffc1b1b6ea9c32103a1c28130846ff6a6a9ca21b3a5b194', 'r2': b'a083f0da94f921d269cde94a1f351e07049f305e66c0948999671d9a5d210ad2', 'r3': b'6119c3109a490e57d72bf1a01621dfb8fa59844a51c852a30d02a0b66f8fce83', 'r4': b'e75463c9d6bbdfc1aacdec956881dc4f9e60987e9b06880becdc9a5e98268f1c'}, 'public_keys': {'o1': b'de059ddde848b16827bda3db0f6dc790b09b15b6fa6caa3acf7641d8ff6c40cc', 'c0': b'eba0aac691b814ddd5cdea62da5ff7c8bc078f227ff1a2518c896dcf8019000a', 'c1': b'9d296a6220522527fa045d461c6c5380114c092b650dbbec3a5f5b4d6f162ee4', 'r0': b'f9debe49758e86882377e7d77d3cf6461a99b929eca3ecffc775442fa0ab7a3e', 'r1': b'd0b4e541830d4a078c8f20cf083296182267464f687231e029da09df1a8adf69', 'r2': b'9eb3395c5b744c323f82157538e31dcfa6d369a184ce78e3da60e97a527a78f8', 'r3': b'c210b906a34b06e0b7b4e34889cd668cd4c9664386cb3294dccb3cab923c64d9', 'r4': b'fab5925f063f7224c62daecbfc91b198ebf39cd1875dd6fd929d401127ecf7a5'}}
[2017-11-20 23:36:58,431] olympus.Olympus<Olympus:bb002>:OUTPUT: Initializing cluster with configuration version:  0
[2017-11-20 23:36:59,598] replica.Replica<Replica:5c003>:OUTPUT: Spawned and initialized replica:  r0
[2017-11-20 23:36:59,604] replica.Replica<Replica:5c004>:OUTPUT: Spawned and initialized replica:  r1
[2017-11-20 23:36:59,609] replica.Replica<Replica:5c005>:OUTPUT: Spawned and initialized replica:  r2
[2017-11-20 23:36:59,614] replica.Replica<Replica:5c006>:OUTPUT: Spawned and initialized replica:  r3
[2017-11-20 23:36:59,620] replica.Replica<Replica:5c007>:OUTPUT: Spawned and initialized replica:  r4
[2017-11-20 23:36:59,626] client.Client<Client:5c001>:OUTPUT: c0  initialized with config= {'id': 'c0', 'workload': ["put('movie','star')", "get('movie')", "append('movie', '123')", "get('movie')", "get('movie');"], 'private_key': b'09e0ab577e1adcdc0d6dbab28ed9bd68086700eb38741f715a0aa6c3bd88fda6', 'public_keys': {'o1': b'de059ddde848b16827bda3db0f6dc790b09b15b6fa6caa3acf7641d8ff6c40cc', 'c0': b'eba0aac691b814ddd5cdea62da5ff7c8bc078f227ff1a2518c896dcf8019000a', 'c1': b'9d296a6220522527fa045d461c6c5380114c092b650dbbec3a5f5b4d6f162ee4', 'r0': b'f9debe49758e86882377e7d77d3cf6461a99b929eca3ecffc775442fa0ab7a3e', 'r1': b'd0b4e541830d4a078c8f20cf083296182267464f687231e029da09df1a8adf69', 'r2': b'9eb3395c5b744c323f82157538e31dcfa6d369a184ce78e3da60e97a527a78f8', 'r3': b'c210b906a34b06e0b7b4e34889cd668cd4c9664386cb3294dccb3cab923c64d9', 'r4': b'fab5925f063f7224c62daecbfc91b198ebf39cd1875dd6fd929d401127ecf7a5'}, 'client_timeout': 3.0, 'olympus': {'id': 'o1', 'process_id': <Olympus:bb002>}, 'replicas': {'r0': <Replica:5c003>, 'r1': <Replica:5c004>, 'r2': <Replica:5c005>, 'r3': <Replica:5c006>, 'r4': <Replica:5c007>}, 'head_id': 'r0', 't': 2}
[2017-11-20 23:36:59,628] client.Client<Client:5c001>:OUTPUT: The following operation was created: {'id': 0, 'client_id': 'c0', 'command_string': "put('movie','star')"}
[2017-11-20 23:36:59,629] client.Client<Client:5c001>:OUTPUT: return_value= OK
[2017-11-20 23:36:59,633] client.Client<Client:5c002>:OUTPUT: c1  initialized with config= {'id': 'c1', 'workload': ["put('movie1','star1')", "get('movie1')", "append('movie1', '123')", "get('movie1')", "get('movie1');"], 'private_key': b'bdcf34d945513f550c3ad755ecf681f1da99b0021a14ff783e06090453ab47c8', 'public_keys': {'o1': b'de059ddde848b16827bda3db0f6dc790b09b15b6fa6caa3acf7641d8ff6c40cc', 'c0': b'eba0aac691b814ddd5cdea62da5ff7c8bc078f227ff1a2518c896dcf8019000a', 'c1': b'9d296a6220522527fa045d461c6c5380114c092b650dbbec3a5f5b4d6f162ee4', 'r0': b'f9debe49758e86882377e7d77d3cf6461a99b929eca3ecffc775442fa0ab7a3e', 'r1': b'd0b4e541830d4a078c8f20cf083296182267464f687231e029da09df1a8adf69', 'r2': b'9eb3395c5b744c323f82157538e31dcfa6d369a184ce78e3da60e97a527a78f8', 'r3': b'c210b906a34b06e0b7b4e34889cd668cd4c9664386cb3294dccb3cab923c64d9', 'r4': b'fab5925f063f7224c62daecbfc91b198ebf39cd1875dd6fd929d401127ecf7a5'}, 'client_timeout': 3.0, 'olympus': {'id': 'o1', 'process_id': <Olympus:bb002>}, 'replicas': {'r0': <Replica:5c003>, 'r1': <Replica:5c004>, 'r2': <Replica:5c005>, 'r3': <Replica:5c006>, 'r4': <Replica:5c007>}, 'head_id': 'r0', 't': 2}
[2017-11-20 23:36:59,635] client.Client<Client:5c001>:OUTPUT: A request for an operation was made {'content': {'is_retry': False, 'operation': {'id': 0, 'client_id': 'c0', 'command_string': "put('movie','star')"}}, 'signed_content': b"\x82\xd2\xf7\r\xb4\x13I\x8d3.=\x91\x8c\x10\x89w\xea[\x14\xc4\x94X\xf2\xa9\x8cp\n6\xd0MT\x9f~\xc0\xf7\xf1\xcbkrb\x04\xcf\xa3+U1~\xf2\xc0\xcf\xb1\xcd \xa5\xa82U)Z\x01\x99\x87#\x06\x80\x03}q\x00(X\x08\x00\x00\x00is_retryq\x01\x89X\t\x00\x00\x00operationq\x02}q\x03(X\x02\x00\x00\x00idq\x04K\x00X\t\x00\x00\x00client_idq\x05X\x02\x00\x00\x00c0q\x06X\x0e\x00\x00\x00command_stringq\x07X\x13\x00\x00\x00put('movie','star')q\x08uu."}
[2017-11-20 23:36:59,637] replica.Replica<Replica:5c003>:OUTPUT: Received request message (source, client_id, operation_id, self_id)  c0 c0 0 r0
[2017-11-20 23:36:59,637] replica.Replica<Replica:5c003>:OUTPUT: Checking for failure associated with trigger, application:  ('client_request', 'pre')  in replica:  r0
[2017-11-20 23:36:59,638] replica.Replica<Replica:5c003>:OUTPUT: No failure associated with trigger:  client_request  found in replica:  r0
[2017-11-20 23:36:59,639] client.Client<Client:5c002>:OUTPUT: The following operation was created: {'id': 0, 'client_id': 'c1', 'command_string': "put('movie1','star1')"}
[2017-11-20 23:36:59,639] client.Client<Client:5c002>:OUTPUT: return_value= OK
[2017-11-20 23:36:59,641] replica.Replica<Replica:5c003>:OUTPUT: Head is processing request (operation_id, client_id) (0, 'c0')
[2017-11-20 23:36:59,643] client.Client<Client:5c002>:OUTPUT: A request for an operation was made {'content': {'is_retry': False, 'operation': {'id': 0, 'client_id': 'c1', 'command_string': "put('movie1','star1')"}}, 'signed_content': b"\xa6M)#\x8b(\xe0|\xa2\xe7k\xcc/ZR\xeb1\xd1\x9c\xc9\xaa\x7f\xfeiB\xcb\xbb\xe4\x13_\x9d@J\x9d\x9e\xd0\xd9\x95\xc2\x1dG\xde~\x19p\x18\xe6p-QZ\xd2\xf4\x05Wn\xae\x0bQ<\xd41\x8f\n\x80\x03}q\x00(X\x08\x00\x00\x00is_retryq\x01\x89X\t\x00\x00\x00operationq\x02}q\x03(X\x02\x00\x00\x00idq\x04K\x00X\t\x00\x00\x00client_idq\x05X\x02\x00\x00\x00c1q\x06X\x0e\x00\x00\x00command_stringq\x07X\x15\x00\x00\x00put('movie1','star1')q\x08uu."}
[2017-11-20 23:36:59,647] replica.Replica<Replica:5c003>:OUTPUT: Checking for failure associated with trigger, application:  ('client_request', 'post')  in replica:  r0
[2017-11-20 23:36:59,648] replica.Replica<Replica:5c003>:OUTPUT: No failure associated with trigger:  client_request  found in replica:  r0
[2017-11-20 23:36:59,652] replica.Replica<Replica:5c003>:OUTPUT: Received request message (source, client_id, operation_id, self_id)  c1 c1 0 r0
[2017-11-20 23:36:59,653] replica.Replica<Replica:5c003>:OUTPUT: Checking for failure associated with trigger, application:  ('client_request', 'pre')  in replica:  r0
[2017-11-20 23:36:59,653] replica.Replica<Replica:5c004>:OUTPUT: Received shuttle (source, client_id, operation_id, self_id)  r0 c0 0 r1
[2017-11-20 23:36:59,653] replica.Replica<Replica:5c003>:OUTPUT: No failure associated with trigger:  client_request  found in replica:  r0
[2017-11-20 23:36:59,654] replica.Replica<Replica:5c004>:OUTPUT: Checking for failure associated with trigger, application:  ('shuttle', 'pre')  in replica:  r1
[2017-11-20 23:36:59,655] replica.Replica<Replica:5c004>:OUTPUT: No failure associated with trigger:  shuttle  found in replica:  r1
[2017-11-20 23:36:59,657] replica.Replica<Replica:5c003>:OUTPUT: Head is processing request (operation_id, client_id) (0, 'c1')
[2017-11-20 23:36:59,664] replica.Replica<Replica:5c003>:OUTPUT: Checking for failure associated with trigger, application:  ('client_request', 'post')  in replica:  r0
[2017-11-20 23:36:59,665] replica.Replica<Replica:5c003>:OUTPUT: No failure associated with trigger:  client_request  found in replica:  r0
[2017-11-20 23:36:59,670] replica.Replica<Replica:5c004>:OUTPUT: Checking for failure associated with trigger, application:  ('shuttle', 'post')  in replica:  r1
[2017-11-20 23:36:59,670] replica.Replica<Replica:5c004>:OUTPUT: No failure associated with trigger:  shuttle  found in replica:  r1
[2017-11-20 23:36:59,674] replica.Replica<Replica:5c004>:OUTPUT: Received shuttle (source, client_id, operation_id, self_id)  r0 c1 0 r1
[2017-11-20 23:36:59,675] replica.Replica<Replica:5c004>:OUTPUT: Checking for failure associated with trigger, application:  ('shuttle', 'pre')  in replica:  r1
[2017-11-20 23:36:59,675] replica.Replica<Replica:5c005>:OUTPUT: Received shuttle (source, client_id, operation_id, self_id)  r1 c0 0 r2
[2017-11-20 23:36:59,675] replica.Replica<Replica:5c004>:OUTPUT: No failure associated with trigger:  shuttle  found in replica:  r1
[2017-11-20 23:36:59,675] replica.Replica<Replica:5c005>:OUTPUT: Checking for failure associated with trigger, application:  ('shuttle', 'pre')  in replica:  r2
[2017-11-20 23:36:59,676] replica.Replica<Replica:5c005>:OUTPUT: No failure associated with trigger:  shuttle  found in replica:  r2
[2017-11-20 23:36:59,692] replica.Replica<Replica:5c004>:OUTPUT: Checking for failure associated with trigger, application:  ('shuttle', 'post')  in replica:  r1
[2017-11-20 23:36:59,692] replica.Replica<Replica:5c004>:OUTPUT: No failure associated with trigger:  shuttle  found in replica:  r1
[2017-11-20 23:36:59,697] replica.Replica<Replica:5c005>:OUTPUT: Checking for failure associated with trigger, application:  ('shuttle', 'post')  in replica:  r2
[2017-11-20 23:36:59,697] replica.Replica<Replica:5c005>:OUTPUT: No failure associated with trigger:  shuttle  found in replica:  r2
[2017-11-20 23:36:59,703] replica.Replica<Replica:5c005>:OUTPUT: Received shuttle (source, client_id, operation_id, self_id)  r1 c1 0 r2
[2017-11-20 23:36:59,704] replica.Replica<Replica:5c005>:OUTPUT: Checking for failure associated with trigger, application:  ('shuttle', 'pre')  in replica:  r2
[2017-11-20 23:36:59,704] replica.Replica<Replica:5c006>:OUTPUT: Received shuttle (source, client_id, operation_id, self_id)  r2 c0 0 r3
[2017-11-20 23:36:59,704] replica.Replica<Replica:5c005>:OUTPUT: No failure associated with trigger:  shuttle  found in replica:  r2
[2017-11-20 23:36:59,704] replica.Replica<Replica:5c006>:OUTPUT: Checking for failure associated with trigger, application:  ('shuttle', 'pre')  in replica:  r3
[2017-11-20 23:36:59,705] replica.Replica<Replica:5c006>:OUTPUT: No failure associated with trigger:  shuttle  found in replica:  r3
[2017-11-20 23:36:59,724] replica.Replica<Replica:5c005>:OUTPUT: Checking for failure associated with trigger, application:  ('shuttle', 'post')  in replica:  r2
[2017-11-20 23:36:59,724] replica.Replica<Replica:5c005>:OUTPUT: No failure associated with trigger:  shuttle  found in replica:  r2
[2017-11-20 23:36:59,727] replica.Replica<Replica:5c006>:OUTPUT: Checking for failure associated with trigger, application:  ('shuttle', 'post')  in replica:  r3
[2017-11-20 23:36:59,728] replica.Replica<Replica:5c006>:OUTPUT: No failure associated with trigger:  shuttle  found in replica:  r3
[2017-11-20 23:36:59,732] replica.Replica<Replica:5c006>:OUTPUT: Received shuttle (source, client_id, operation_id, self_id)  r2 c1 0 r3
[2017-11-20 23:36:59,733] replica.Replica<Replica:5c007>:OUTPUT: Received shuttle (source, client_id, operation_id, self_id)  r3 c0 0 r4
[2017-11-20 23:36:59,733] replica.Replica<Replica:5c006>:OUTPUT: Checking for failure associated with trigger, application:  ('shuttle', 'pre')  in replica:  r3
[2017-11-20 23:36:59,734] replica.Replica<Replica:5c007>:OUTPUT: Checking for failure associated with trigger, application:  ('shuttle', 'pre')  in replica:  r4
[2017-11-20 23:36:59,734] replica.Replica<Replica:5c006>:OUTPUT: No failure associated with trigger:  shuttle  found in replica:  r3
[2017-11-20 23:36:59,735] replica.Replica<Replica:5c007>:OUTPUT: No failure associated with trigger:  shuttle  found in replica:  r4
[2017-11-20 23:36:59,756] replica.Replica<Replica:5c006>:OUTPUT: Checking for failure associated with trigger, application:  ('shuttle', 'post')  in replica:  r3
[2017-11-20 23:36:59,756] replica.Replica<Replica:5c006>:OUTPUT: No failure associated with trigger:  shuttle  found in replica:  r3
[2017-11-20 23:36:59,774] replica.Replica<Replica:5c007>:OUTPUT: Checking for failure associated with trigger, application:  ('shuttle', 'post')  in replica:  r4
[2017-11-20 23:36:59,774] replica.Replica<Replica:5c007>:OUTPUT: No failure associated with trigger:  shuttle  found in replica:  r4
[2017-11-20 23:36:59,780] replica.Replica<Replica:5c006>:OUTPUT: Received result shuttle (source, client_id, operation_id, self_id)  r4 c0 0 r3
[2017-11-20 23:36:59,780] replica.Replica<Replica:5c007>:OUTPUT: Checking for failure associated with trigger, application:  ('shuttle', 'post')  in replica:  r4
[2017-11-20 23:36:59,780] replica.Replica<Replica:5c006>:OUTPUT: Checking for failure associated with trigger, application:  ('result_shuttle', 'pre')  in replica:  r3
[2017-11-20 23:36:59,780] replica.Replica<Replica:5c007>:OUTPUT: No failure associated with trigger:  shuttle  found in replica:  r4
[2017-11-20 23:36:59,781] replica.Replica<Replica:5c006>:OUTPUT: No failure associated with trigger:  result_shuttle  found in replica:  r3
[2017-11-20 23:36:59,786] replica.Replica<Replica:5c007>:OUTPUT: Received shuttle (source, client_id, operation_id, self_id)  r3 c1 0 r4
[2017-11-20 23:36:59,786] client.Client<Client:5c001>:OUTPUT: A result was received from replica= r4
[2017-11-20 23:36:59,786] replica.Replica<Replica:5c007>:OUTPUT: Checking for failure associated with trigger, application:  ('shuttle', 'pre')  in replica:  r4
[2017-11-20 23:36:59,787] replica.Replica<Replica:5c007>:OUTPUT: No failure associated with trigger:  shuttle  found in replica:  r4
[2017-11-20 23:36:59,807] replica.Replica<Replica:5c006>:OUTPUT: Checking for failure associated with trigger, application:  ('result_shuttle', 'post')  in replica:  r3
[2017-11-20 23:36:59,807] replica.Replica<Replica:5c006>:OUTPUT: No failure associated with trigger:  result_shuttle  found in replica:  r3
[2017-11-20 23:36:59,808] client.Client<Client:5c001>:OUTPUT: The operation result pair is= {'id': 0, 'client_id': 'c0', 'command_string': "put('movie','star')"} OK
[2017-11-20 23:36:59,809] client.Client<Client:5c001>:OUTPUT: The result received is valid:(result, operation)= OK {'id': 0, 'client_id': 'c0', 'command_string': "put('movie','star')"}
[2017-11-20 23:36:59,810] client.Client<Client:5c001>:OUTPUT: The following operation was created: {'id': 1, 'client_id': 'c0', 'command_string': "get('movie')"}
[2017-11-20 23:36:59,810] client.Client<Client:5c001>:OUTPUT: return_value= star
[2017-11-20 23:36:59,813] client.Client<Client:5c001>:OUTPUT: A request for an operation was made {'content': {'is_retry': False, 'operation': {'id': 1, 'client_id': 'c0', 'command_string': "get('movie')"}}, 'signed_content': b'g\n/\x06\xdf\x1a\xbf\x1bs\xddYbnD\xe0=\xa3,\x9f\x10W\x97\xbd\xc2\xd8\xa4\x182\x9d\x1czrb\x85"2\xee\xc2]6\xb3\x15B\x06\xf9\x8a\xafq\x8d\x19\xfe\x81O\xb0\x07X]\x8a\xc87\xf0\xc2\xe4\x04\x80\x03}q\x00(X\x08\x00\x00\x00is_retryq\x01\x89X\t\x00\x00\x00operationq\x02}q\x03(X\x02\x00\x00\x00idq\x04K\x01X\t\x00\x00\x00client_idq\x05X\x02\x00\x00\x00c0q\x06X\x0e\x00\x00\x00command_stringq\x07X\x0c\x00\x00\x00get(\'movie\')q\x08uu.'}
[2017-11-20 23:36:59,814] replica.Replica<Replica:5c003>:OUTPUT: Received request message (source, client_id, operation_id, self_id)  c0 c0 1 r0
[2017-11-20 23:36:59,815] replica.Replica<Replica:5c003>:OUTPUT: Checking for failure associated with trigger, application:  ('client_request', 'pre')  in replica:  r0
[2017-11-20 23:36:59,815] replica.Replica<Replica:5c005>:OUTPUT: Received result shuttle (source, client_id, operation_id, self_id)  r3 c0 0 r2
[2017-11-20 23:36:59,815] replica.Replica<Replica:5c003>:OUTPUT: No failure associated with trigger:  client_request  found in replica:  r0
[2017-11-20 23:36:59,815] replica.Replica<Replica:5c005>:OUTPUT: Checking for failure associated with trigger, application:  ('result_shuttle', 'pre')  in replica:  r2
[2017-11-20 23:36:59,816] replica.Replica<Replica:5c005>:OUTPUT: No failure associated with trigger:  result_shuttle  found in replica:  r2
[2017-11-20 23:36:59,820] replica.Replica<Replica:5c003>:OUTPUT: Head is processing request (operation_id, client_id) (1, 'c0')
[2017-11-20 23:36:59,827] replica.Replica<Replica:5c003>:OUTPUT: Checking for failure associated with trigger, application:  ('client_request', 'post')  in replica:  r0
[2017-11-20 23:36:59,827] replica.Replica<Replica:5c003>:OUTPUT: No failure associated with trigger:  client_request  found in replica:  r0
[2017-11-20 23:36:59,830] replica.Replica<Replica:5c004>:OUTPUT: Received shuttle (source, client_id, operation_id, self_id)  r0 c0 1 r1
[2017-11-20 23:36:59,831] replica.Replica<Replica:5c004>:OUTPUT: Checking for failure associated with trigger, application:  ('shuttle', 'pre')  in replica:  r1
[2017-11-20 23:36:59,831] replica.Replica<Replica:5c004>:OUTPUT: No failure associated with trigger:  shuttle  found in replica:  r1
[2017-11-20 23:36:59,837] replica.Replica<Replica:5c007>:OUTPUT: Checking for failure associated with trigger, application:  ('shuttle', 'post')  in replica:  r4
[2017-11-20 23:36:59,838] replica.Replica<Replica:5c007>:OUTPUT: No failure associated with trigger:  shuttle  found in replica:  r4
[2017-11-20 23:36:59,839] replica.Replica<Replica:5c005>:OUTPUT: Checking for failure associated with trigger, application:  ('result_shuttle', 'post')  in replica:  r2
[2017-11-20 23:36:59,839] replica.Replica<Replica:5c005>:OUTPUT: No failure associated with trigger:  result_shuttle  found in replica:  r2
[2017-11-20 23:36:59,844] replica.Replica<Replica:5c006>:OUTPUT: Received result shuttle (source, client_id, operation_id, self_id)  r4 c1 0 r3
[2017-11-20 23:36:59,845] replica.Replica<Replica:5c006>:OUTPUT: Checking for failure associated with trigger, application:  ('result_shuttle', 'pre')  in replica:  r3
[2017-11-20 23:36:59,846] replica.Replica<Replica:5c006>:OUTPUT: No failure associated with trigger:  result_shuttle  found in replica:  r3
[2017-11-20 23:36:59,847] replica.Replica<Replica:5c007>:OUTPUT: Checking for failure associated with trigger, application:  ('shuttle', 'post')  in replica:  r4
[2017-11-20 23:36:59,848] replica.Replica<Replica:5c007>:OUTPUT: No failure associated with trigger:  shuttle  found in replica:  r4
[2017-11-20 23:36:59,849] replica.Replica<Replica:5c004>:OUTPUT: Checking for failure associated with trigger, application:  ('shuttle', 'post')  in replica:  r1
[2017-11-20 23:36:59,850] replica.Replica<Replica:5c004>:OUTPUT: No failure associated with trigger:  shuttle  found in replica:  r1
[2017-11-20 23:36:59,853] client.Client<Client:5c002>:OUTPUT: A result was received from replica= r4
[2017-11-20 23:36:59,854] replica.Replica<Replica:5c005>:OUTPUT: Received shuttle (source, client_id, operation_id, self_id)  r1 c0 1 r2
[2017-11-20 23:36:59,854] replica.Replica<Replica:5c004>:OUTPUT: Received result shuttle (source, client_id, operation_id, self_id)  r2 c0 0 r1
[2017-11-20 23:36:59,855] replica.Replica<Replica:5c005>:OUTPUT: Checking for failure associated with trigger, application:  ('shuttle', 'pre')  in replica:  r2
[2017-11-20 23:36:59,855] replica.Replica<Replica:5c004>:OUTPUT: Checking for failure associated with trigger, application:  ('result_shuttle', 'pre')  in replica:  r1
[2017-11-20 23:36:59,855] replica.Replica<Replica:5c005>:OUTPUT: No failure associated with trigger:  shuttle  found in replica:  r2
[2017-11-20 23:36:59,855] replica.Replica<Replica:5c004>:OUTPUT: No failure associated with trigger:  result_shuttle  found in replica:  r1
[2017-11-20 23:36:59,877] client.Client<Client:5c002>:OUTPUT: The operation result pair is= {'id': 0, 'client_id': 'c1', 'command_string': "put('movie1','star1')"} OK
[2017-11-20 23:36:59,877] replica.Replica<Replica:5c005>:OUTPUT: Checking for failure associated with trigger, application:  ('shuttle', 'post')  in replica:  r2
[2017-11-20 23:36:59,876] replica.Replica<Replica:5c006>:OUTPUT: Checking for failure associated with trigger, application:  ('result_shuttle', 'post')  in replica:  r3
[2017-11-20 23:36:59,877] client.Client<Client:5c002>:OUTPUT: The result received is valid:(result, operation)= OK {'id': 0, 'client_id': 'c1', 'command_string': "put('movie1','star1')"}
[2017-11-20 23:36:59,878] replica.Replica<Replica:5c005>:OUTPUT: No failure associated with trigger:  shuttle  found in replica:  r2
[2017-11-20 23:36:59,878] client.Client<Client:5c002>:OUTPUT: The following operation was created: {'id': 1, 'client_id': 'c1', 'command_string': "get('movie1')"}
[2017-11-20 23:36:59,878] client.Client<Client:5c002>:OUTPUT: return_value= star1
[2017-11-20 23:36:59,878] replica.Replica<Replica:5c006>:OUTPUT: No failure associated with trigger:  result_shuttle  found in replica:  r3
[2017-11-20 23:36:59,881] client.Client<Client:5c002>:OUTPUT: A request for an operation was made {'content': {'is_retry': False, 'operation': {'id': 1, 'client_id': 'c1', 'command_string': "get('movie1')"}}, 'signed_content': b'l\x90p\xf1\x8f:X\x1e\xc2\x06\xad".\xeesq\xd8\xda\xea\xfe\xf3\x03\xc8\xfd\xf9\xfcv\nq&R\xabdG\xdbW\xfa\xe9\xd5\xc1\xcaA\xbaUf\xf1s\xce\xa0T=\xa8/\x99>\x11\x1d\x92wB5w\x18\x07\x80\x03}q\x00(X\x08\x00\x00\x00is_retryq\x01\x89X\t\x00\x00\x00operationq\x02}q\x03(X\x02\x00\x00\x00idq\x04K\x01X\t\x00\x00\x00client_idq\x05X\x02\x00\x00\x00c1q\x06X\x0e\x00\x00\x00command_stringq\x07X\r\x00\x00\x00get(\'movie1\')q\x08uu.'}
[2017-11-20 23:36:59,882] replica.Replica<Replica:5c003>:OUTPUT: Received request message (source, client_id, operation_id, self_id)  c1 c1 1 r0
[2017-11-20 23:36:59,883] replica.Replica<Replica:5c003>:OUTPUT: Checking for failure associated with trigger, application:  ('client_request', 'pre')  in replica:  r0
[2017-11-20 23:36:59,883] replica.Replica<Replica:5c003>:OUTPUT: No failure associated with trigger:  client_request  found in replica:  r0
[2017-11-20 23:36:59,885] replica.Replica<Replica:5c005>:OUTPUT: Received result shuttle (source, client_id, operation_id, self_id)  r3 c1 0 r2
[2017-11-20 23:36:59,886] replica.Replica<Replica:5c005>:OUTPUT: Checking for failure associated with trigger, application:  ('result_shuttle', 'pre')  in replica:  r2
[2017-11-20 23:36:59,887] replica.Replica<Replica:5c005>:OUTPUT: No failure associated with trigger:  result_shuttle  found in replica:  r2
[2017-11-20 23:36:59,888] replica.Replica<Replica:5c003>:OUTPUT: Head is processing request (operation_id, client_id) (1, 'c1')
[2017-11-20 23:36:59,885] replica.Replica<Replica:5c006>:OUTPUT: Received shuttle (source, client_id, operation_id, self_id)  r2 c0 1 r3
[2017-11-20 23:36:59,891] replica.Replica<Replica:5c006>:OUTPUT: Checking for failure associated with trigger, application:  ('shuttle', 'pre')  in replica:  r3
[2017-11-20 23:36:59,890] replica.Replica<Replica:5c004>:OUTPUT: Checking for failure associated with trigger, application:  ('result_shuttle', 'post')  in replica:  r1
[2017-11-20 23:36:59,892] replica.Replica<Replica:5c006>:OUTPUT: No failure associated with trigger:  shuttle  found in replica:  r3
[2017-11-20 23:36:59,892] replica.Replica<Replica:5c004>:OUTPUT: No failure associated with trigger:  result_shuttle  found in replica:  r1
[2017-11-20 23:36:59,895] replica.Replica<Replica:5c003>:OUTPUT: Checking for failure associated with trigger, application:  ('client_request', 'post')  in replica:  r0
[2017-11-20 23:36:59,896] replica.Replica<Replica:5c003>:OUTPUT: No failure associated with trigger:  client_request  found in replica:  r0
[2017-11-20 23:36:59,901] replica.Replica<Replica:5c004>:OUTPUT: Received shuttle (source, client_id, operation_id, self_id)  r0 c1 1 r1
[2017-11-20 23:36:59,902] replica.Replica<Replica:5c004>:OUTPUT: Checking for failure associated with trigger, application:  ('shuttle', 'pre')  in replica:  r1
[2017-11-20 23:36:59,902] replica.Replica<Replica:5c004>:OUTPUT: No failure associated with trigger:  shuttle  found in replica:  r1
[2017-11-20 23:36:59,903] replica.Replica<Replica:5c003>:OUTPUT: Received result shuttle (source, client_id, operation_id, self_id)  r1 c0 0 r0
[2017-11-20 23:36:59,903] replica.Replica<Replica:5c003>:OUTPUT: Checking for failure associated with trigger, application:  ('result_shuttle', 'pre')  in replica:  r0
[2017-11-20 23:36:59,903] replica.Replica<Replica:5c003>:OUTPUT: No failure associated with trigger:  result_shuttle  found in replica:  r0
[2017-11-20 23:36:59,921] replica.Replica<Replica:5c004>:OUTPUT: Checking for failure associated with trigger, application:  ('shuttle', 'post')  in replica:  r1
[2017-11-20 23:36:59,921] replica.Replica<Replica:5c004>:OUTPUT: No failure associated with trigger:  shuttle  found in replica:  r1
[2017-11-20 23:36:59,923] replica.Replica<Replica:5c006>:OUTPUT: Checking for failure associated with trigger, application:  ('shuttle', 'post')  in replica:  r3
[2017-11-20 23:36:59,924] replica.Replica<Replica:5c006>:OUTPUT: No failure associated with trigger:  shuttle  found in replica:  r3
[2017-11-20 23:36:59,927] replica.Replica<Replica:5c005>:OUTPUT: Checking for failure associated with trigger, application:  ('result_shuttle', 'post')  in replica:  r2
[2017-11-20 23:36:59,927] replica.Replica<Replica:5c005>:OUTPUT: No failure associated with trigger:  result_shuttle  found in replica:  r2
[2017-11-20 23:36:59,929] replica.Replica<Replica:5c007>:OUTPUT: Received shuttle (source, client_id, operation_id, self_id)  r3 c0 1 r4
[2017-11-20 23:36:59,930] replica.Replica<Replica:5c007>:OUTPUT: Checking for failure associated with trigger, application:  ('shuttle', 'pre')  in replica:  r4
[2017-11-20 23:36:59,930] replica.Replica<Replica:5c007>:OUTPUT: No failure associated with trigger:  shuttle  found in replica:  r4
[2017-11-20 23:36:59,931] replica.Replica<Replica:5c005>:OUTPUT: Received shuttle (source, client_id, operation_id, self_id)  r1 c1 1 r2
[2017-11-20 23:36:59,932] replica.Replica<Replica:5c005>:OUTPUT: Checking for failure associated with trigger, application:  ('shuttle', 'pre')  in replica:  r2
[2017-11-20 23:36:59,932] replica.Replica<Replica:5c005>:OUTPUT: No failure associated with trigger:  shuttle  found in replica:  r2
[2017-11-20 23:36:59,933] replica.Replica<Replica:5c004>:OUTPUT: Received result shuttle (source, client_id, operation_id, self_id)  r2 c1 0 r1
[2017-11-20 23:36:59,933] replica.Replica<Replica:5c004>:OUTPUT: Checking for failure associated with trigger, application:  ('result_shuttle', 'pre')  in replica:  r1
[2017-11-20 23:36:59,933] replica.Replica<Replica:5c004>:OUTPUT: No failure associated with trigger:  result_shuttle  found in replica:  r1
[2017-11-20 23:36:59,959] replica.Replica<Replica:5c005>:OUTPUT: Checking for failure associated with trigger, application:  ('shuttle', 'post')  in replica:  r2
[2017-11-20 23:36:59,959] replica.Replica<Replica:5c005>:OUTPUT: No failure associated with trigger:  shuttle  found in replica:  r2
[2017-11-20 23:36:59,960] replica.Replica<Replica:5c004>:OUTPUT: Checking for failure associated with trigger, application:  ('result_shuttle', 'post')  in replica:  r1
[2017-11-20 23:36:59,960] replica.Replica<Replica:5c004>:OUTPUT: No failure associated with trigger:  result_shuttle  found in replica:  r1
[2017-11-20 23:36:59,963] replica.Replica<Replica:5c006>:OUTPUT: Received shuttle (source, client_id, operation_id, self_id)  r2 c1 1 r3
[2017-11-20 23:36:59,964] replica.Replica<Replica:5c006>:OUTPUT: Checking for failure associated with trigger, application:  ('shuttle', 'pre')  in replica:  r3
[2017-11-20 23:36:59,964] replica.Replica<Replica:5c006>:OUTPUT: No failure associated with trigger:  shuttle  found in replica:  r3
[2017-11-20 23:36:59,965] replica.Replica<Replica:5c003>:OUTPUT: Received result shuttle (source, client_id, operation_id, self_id)  r1 c1 0 r0
[2017-11-20 23:36:59,966] replica.Replica<Replica:5c003>:OUTPUT: Checking for failure associated with trigger, application:  ('result_shuttle', 'pre')  in replica:  r0
[2017-11-20 23:36:59,966] replica.Replica<Replica:5c003>:OUTPUT: No failure associated with trigger:  result_shuttle  found in replica:  r0
[2017-11-20 23:36:59,981] replica.Replica<Replica:5c007>:OUTPUT: Checking for failure associated with trigger, application:  ('shuttle', 'post')  in replica:  r4
[2017-11-20 23:36:59,981] replica.Replica<Replica:5c007>:OUTPUT: No failure associated with trigger:  shuttle  found in replica:  r4
[2017-11-20 23:36:59,988] replica.Replica<Replica:5c006>:OUTPUT: Checking for failure associated with trigger, application:  ('shuttle', 'post')  in replica:  r3
[2017-11-20 23:36:59,988] replica.Replica<Replica:5c006>:OUTPUT: No failure associated with trigger:  shuttle  found in replica:  r3
[2017-11-20 23:36:59,989] replica.Replica<Replica:5c007>:OUTPUT: Checking for failure associated with trigger, application:  ('shuttle', 'post')  in replica:  r4
[2017-11-20 23:36:59,991] replica.Replica<Replica:5c007>:OUTPUT: No failure associated with trigger:  shuttle  found in replica:  r4
[2017-11-20 23:36:59,993] replica.Replica<Replica:5c006>:OUTPUT: Received result shuttle (source, client_id, operation_id, self_id)  r4 c0 1 r3
[2017-11-20 23:36:59,993] replica.Replica<Replica:5c006>:OUTPUT: Checking for failure associated with trigger, application:  ('result_shuttle', 'pre')  in replica:  r3
[2017-11-20 23:36:59,994] replica.Replica<Replica:5c006>:OUTPUT: No failure associated with trigger:  result_shuttle  found in replica:  r3
[2017-11-20 23:36:59,995] client.Client<Client:5c001>:OUTPUT: A result was received from replica= r4
[2017-11-20 23:36:59,996] replica.Replica<Replica:5c007>:OUTPUT: Received shuttle (source, client_id, operation_id, self_id)  r3 c1 1 r4
[2017-11-20 23:36:59,996] replica.Replica<Replica:5c007>:OUTPUT: Checking for failure associated with trigger, application:  ('shuttle', 'pre')  in replica:  r4
[2017-11-20 23:36:59,997] replica.Replica<Replica:5c007>:OUTPUT: No failure associated with trigger:  shuttle  found in replica:  r4
[2017-11-20 23:37:00,016] client.Client<Client:5c001>:OUTPUT: The operation result pair is= {'id': 1, 'client_id': 'c0', 'command_string': "get('movie')"} star
[2017-11-20 23:37:00,017] client.Client<Client:5c001>:OUTPUT: The result received is valid:(result, operation)= star {'id': 1, 'client_id': 'c0', 'command_string': "get('movie')"}
[2017-11-20 23:37:00,017] client.Client<Client:5c001>:OUTPUT: The following operation was created: {'id': 2, 'client_id': 'c0', 'command_string': "append('movie', '123')"}
[2017-11-20 23:37:00,018] client.Client<Client:5c001>:OUTPUT: return_value= OK
[2017-11-20 23:37:00,021] client.Client<Client:5c001>:OUTPUT: A request for an operation was made {'content': {'is_retry': False, 'operation': {'id': 2, 'client_id': 'c0', 'command_string': "append('movie', '123')"}}, 'signed_content': b"6\xe7:\xed\xc3\xbc\x1a\x8d\x81Z\xca\xe3z\xda\x84\x7f\xf01]\xa5\xb4\xb9\xa8X8v\x8d|NvS\x8c\xfd\x02\xe1_){K\x92\xa8\x0e\xf34A\xca\xfd\xd7e\xaaM\x19\xb3\xcd\x05\x9c\xe7\x0e9K\xb7\x05\xa3\x06\x80\x03}q\x00(X\x08\x00\x00\x00is_retryq\x01\x89X\t\x00\x00\x00operationq\x02}q\x03(X\x02\x00\x00\x00idq\x04K\x02X\t\x00\x00\x00client_idq\x05X\x02\x00\x00\x00c0q\x06X\x0e\x00\x00\x00command_stringq\x07X\x16\x00\x00\x00append('movie', '123')q\x08uu."}
[2017-11-20 23:37:00,023] replica.Replica<Replica:5c003>:OUTPUT: Received request message (source, client_id, operation_id, self_id)  c0 c0 2 r0
[2017-11-20 23:37:00,020] replica.Replica<Replica:5c006>:OUTPUT: Checking for failure associated with trigger, application:  ('result_shuttle', 'post')  in replica:  r3
[2017-11-20 23:37:00,023] replica.Replica<Replica:5c003>:OUTPUT: Checking for failure associated with trigger, application:  ('client_request', 'pre')  in replica:  r0
[2017-11-20 23:37:00,023] replica.Replica<Replica:5c006>:OUTPUT: No failure associated with trigger:  result_shuttle  found in replica:  r3
[2017-11-20 23:37:00,023] replica.Replica<Replica:5c003>:OUTPUT: No failure associated with trigger:  client_request  found in replica:  r0
[2017-11-20 23:37:00,028] replica.Replica<Replica:5c003>:OUTPUT: Head is processing request (operation_id, client_id) (2, 'c0')
[2017-11-20 23:37:00,028] replica.Replica<Replica:5c005>:OUTPUT: Received result shuttle (source, client_id, operation_id, self_id)  r3 c0 1 r2
[2017-11-20 23:37:00,029] replica.Replica<Replica:5c005>:OUTPUT: Checking for failure associated with trigger, application:  ('result_shuttle', 'pre')  in replica:  r2
[2017-11-20 23:37:00,029] replica.Replica<Replica:5c005>:OUTPUT: No failure associated with trigger:  result_shuttle  found in replica:  r2
[2017-11-20 23:37:00,036] replica.Replica<Replica:5c003>:OUTPUT: Checking for failure associated with trigger, application:  ('client_request', 'post')  in replica:  r0
[2017-11-20 23:37:00,037] replica.Replica<Replica:5c003>:OUTPUT: No failure associated with trigger:  client_request  found in replica:  r0
[2017-11-20 23:37:00,041] replica.Replica<Replica:5c004>:OUTPUT: Received shuttle (source, client_id, operation_id, self_id)  r0 c0 2 r1
[2017-11-20 23:37:00,041] replica.Replica<Replica:5c004>:OUTPUT: Checking for failure associated with trigger, application:  ('shuttle', 'pre')  in replica:  r1
[2017-11-20 23:37:00,041] replica.Replica<Replica:5c004>:OUTPUT: No failure associated with trigger:  shuttle  found in replica:  r1
[2017-11-20 23:37:00,054] replica.Replica<Replica:5c005>:OUTPUT: Checking for failure associated with trigger, application:  ('result_shuttle', 'post')  in replica:  r2
[2017-11-20 23:37:00,055] replica.Replica<Replica:5c005>:OUTPUT: No failure associated with trigger:  result_shuttle  found in replica:  r2
[2017-11-20 23:37:00,056] replica.Replica<Replica:5c007>:OUTPUT: Checking for failure associated with trigger, application:  ('shuttle', 'post')  in replica:  r4
[2017-11-20 23:37:00,057] replica.Replica<Replica:5c007>:OUTPUT: No failure associated with trigger:  shuttle  found in replica:  r4
[2017-11-20 23:37:00,060] replica.Replica<Replica:5c004>:OUTPUT: Checking for failure associated with trigger, application:  ('shuttle', 'post')  in replica:  r1
[2017-11-20 23:37:00,060] replica.Replica<Replica:5c004>:OUTPUT: No failure associated with trigger:  shuttle  found in replica:  r1
[2017-11-20 23:37:00,063] replica.Replica<Replica:5c006>:OUTPUT: Received result shuttle (source, client_id, operation_id, self_id)  r4 c1 1 r3
[2017-11-20 23:37:00,064] replica.Replica<Replica:5c006>:OUTPUT: Checking for failure associated with trigger, application:  ('result_shuttle', 'pre')  in replica:  r3
[2017-11-20 23:37:00,065] replica.Replica<Replica:5c006>:OUTPUT: No failure associated with trigger:  result_shuttle  found in replica:  r3
[2017-11-20 23:37:00,066] replica.Replica<Replica:5c005>:OUTPUT: Received shuttle (source, client_id, operation_id, self_id)  r1 c0 2 r2
[2017-11-20 23:37:00,064] replica.Replica<Replica:5c007>:OUTPUT: Checking for failure associated with trigger, application:  ('shuttle', 'post')  in replica:  r4
[2017-11-20 23:37:00,067] replica.Replica<Replica:5c005>:OUTPUT: Checking for failure associated with trigger, application:  ('shuttle', 'pre')  in replica:  r2
[2017-11-20 23:37:00,067] replica.Replica<Replica:5c004>:OUTPUT: Received result shuttle (source, client_id, operation_id, self_id)  r2 c0 1 r1
[2017-11-20 23:37:00,067] replica.Replica<Replica:5c007>:OUTPUT: No failure associated with trigger:  shuttle  found in replica:  r4
[2017-11-20 23:37:00,068] replica.Replica<Replica:5c005>:OUTPUT: No failure associated with trigger:  shuttle  found in replica:  r2
[2017-11-20 23:37:00,072] client.Client<Client:5c002>:OUTPUT: A result was received from replica= r4
[2017-11-20 23:37:00,068] replica.Replica<Replica:5c004>:OUTPUT: Checking for failure associated with trigger, application:  ('result_shuttle', 'pre')  in replica:  r1
[2017-11-20 23:37:00,074] replica.Replica<Replica:5c004>:OUTPUT: No failure associated with trigger:  result_shuttle  found in replica:  r1
[2017-11-20 23:37:00,090] replica.Replica<Replica:5c006>:OUTPUT: Checking for failure associated with trigger, application:  ('result_shuttle', 'post')  in replica:  r3
[2017-11-20 23:37:00,091] replica.Replica<Replica:5c006>:OUTPUT: No failure associated with trigger:  result_shuttle  found in replica:  r3
[2017-11-20 23:37:00,094] replica.Replica<Replica:5c005>:OUTPUT: Checking for failure associated with trigger, application:  ('shuttle', 'post')  in replica:  r2
[2017-11-20 23:37:00,094] replica.Replica<Replica:5c005>:OUTPUT: No failure associated with trigger:  shuttle  found in replica:  r2
[2017-11-20 23:37:00,096] client.Client<Client:5c002>:OUTPUT: The operation result pair is= {'id': 1, 'client_id': 'c1', 'command_string': "get('movie1')"} star1
[2017-11-20 23:37:00,096] client.Client<Client:5c002>:OUTPUT: The result received is valid:(result, operation)= star1 {'id': 1, 'client_id': 'c1', 'command_string': "get('movie1')"}
[2017-11-20 23:37:00,097] client.Client<Client:5c002>:OUTPUT: The following operation was created: {'id': 2, 'client_id': 'c1', 'command_string': "append('movie1', '123')"}
[2017-11-20 23:37:00,097] client.Client<Client:5c002>:OUTPUT: return_value= OK
[2017-11-20 23:37:00,098] replica.Replica<Replica:5c005>:OUTPUT: Received result shuttle (source, client_id, operation_id, self_id)  r3 c1 1 r2
[2017-11-20 23:37:00,099] replica.Replica<Replica:5c006>:OUTPUT: Received shuttle (source, client_id, operation_id, self_id)  r2 c0 2 r3
[2017-11-20 23:37:00,099] replica.Replica<Replica:5c005>:OUTPUT: Checking for failure associated with trigger, application:  ('result_shuttle', 'pre')  in replica:  r2
[2017-11-20 23:37:00,099] replica.Replica<Replica:5c006>:OUTPUT: Checking for failure associated with trigger, application:  ('shuttle', 'pre')  in replica:  r3
[2017-11-20 23:37:00,099] replica.Replica<Replica:5c005>:OUTPUT: No failure associated with trigger:  result_shuttle  found in replica:  r2
[2017-11-20 23:37:00,100] client.Client<Client:5c002>:OUTPUT: A request for an operation was made {'content': {'is_retry': False, 'operation': {'id': 2, 'client_id': 'c1', 'command_string': "append('movie1', '123')"}}, 'signed_content': b"\xf3*\x12\xc8mH\xbbJ\x93\r\x04\x00\xf2\x0eO\xcf\xe9\xd1\xeb\xf8+\xd7)\x04\xdb\x88M\xe1@\xb6=\x15\xdc\x1e\x06\x0f\xc5\xd9,\x84\x8f_\xdd\x1d\xd9\x8fcz\t\xe1\xe3\x00$\n\xe2V\xb6w\xe6O'2\xe9\x0b\x80\x03}q\x00(X\x08\x00\x00\x00is_retryq\x01\x89X\t\x00\x00\x00operationq\x02}q\x03(X\x02\x00\x00\x00idq\x04K\x02X\t\x00\x00\x00client_idq\x05X\x02\x00\x00\x00c1q\x06X\x0e\x00\x00\x00command_stringq\x07X\x17\x00\x00\x00append('movie1', '123')q\x08uu."}
[2017-11-20 23:37:00,102] replica.Replica<Replica:5c003>:OUTPUT: Received request message (source, client_id, operation_id, self_id)  c1 c1 2 r0
[2017-11-20 23:37:00,102] replica.Replica<Replica:5c003>:OUTPUT: Checking for failure associated with trigger, application:  ('client_request', 'pre')  in replica:  r0
[2017-11-20 23:37:00,100] replica.Replica<Replica:5c006>:OUTPUT: No failure associated with trigger:  shuttle  found in replica:  r3
[2017-11-20 23:37:00,103] replica.Replica<Replica:5c003>:OUTPUT: No failure associated with trigger:  client_request  found in replica:  r0
[2017-11-20 23:37:00,107] replica.Replica<Replica:5c003>:OUTPUT: Head is processing request (operation_id, client_id) (2, 'c1')
[2017-11-20 23:37:00,103] replica.Replica<Replica:5c004>:OUTPUT: Checking for failure associated with trigger, application:  ('result_shuttle', 'post')  in replica:  r1
[2017-11-20 23:37:00,114] replica.Replica<Replica:5c004>:OUTPUT: No failure associated with trigger:  result_shuttle  found in replica:  r1
', 'post')  in replica:  r0
[2017-11-20 23:37:00,115] replica.Replica<Replica:5c003>:OUTPUT: No failure associated with trigger:  client_request  found in replica:  r0
[2017-11-20 23:37:00,120] replica.Replica<Replica:5c004>:OUTPUT: Received shuttle (source, client_id, operation_id, self_id)  r0 c1 2 r1
[2017-11-20 23:37:00,121] replica.Replica<Replica:5c004>:OUTPUT: Checking for failure associated with trigger, application:  ('shuttle', 'pre')  in replica:  r1
[2017-11-20 23:37:00,121] replica.Replica<Replica:5c004>:OUTPUT: No failure associated with trigger:  shuttle  found in replica:  r1
[2017-11-20 23:37:00,121] replica.Replica<Replica:5c003>:OUTPUT: Received result shuttle (source, client_id, operation_id, self_id)  r1 c0 1 r0
[2017-11-20 23:37:00,122] replica.Replica<Replica:5c003>:OUTPUT: Checking for failure associated with trigger, application:  ('result_shuttle', 'pre')  in replica:  r0
[2017-11-20 23:37:00,122] replica.Replica<Replica:5c003>:OUTPUT: No failure associated with trigger:  result_shuttle  found in replica:  r0
[2017-11-20 23:37:00,131] replica.Replica<Replica:5c006>:OUTPUT: Checking for failure associated with trigger, application:  ('shuttle', 'post')  in replica:  r3
[2017-11-20 23:37:00,133] replica.Replica<Replica:5c005>:OUTPUT: Checking for failure associated with trigger, application:  ('result_shuttle', 'post')  in replica:  r2
[2017-11-20 23:37:00,134] replica.Replica<Replica:5c006>:OUTPUT: No failure associated with trigger:  shuttle  found in replica:  r3
[2017-11-20 23:37:00,134] replica.Replica<Replica:5c005>:OUTPUT: No failure associated with trigger:  result_shuttle  found in replica:  r2
[2017-11-20 23:37:00,139] replica.Replica<Replica:5c007>:OUTPUT: Received shuttle (source, client_id, operation_id, self_id)  r3 c0 2 r4
[2017-11-20 23:37:00,140] replica.Replica<Replica:5c007>:OUTPUT: Checking for failure associated with trigger, application:  ('shuttle', 'pre')  in replica:  r4
[2017-11-20 23:37:00,140] replica.Replica<Replica:5c007>:OUTPUT: No failure associated with trigger:  shuttle  found in replica:  r4
[2017-11-20 23:37:00,144] replica.Replica<Replica:5c004>:OUTPUT: Checking for failure associated with trigger, application:  ('shuttle', 'post')  in replica:  r1
[2017-11-20 23:37:00,145] replica.Replica<Replica:5c004>:OUTPUT: No failure associated with trigger:  shuttle  found in replica:  r1
[2017-11-20 23:37:00,149] replica.Replica<Replica:5c005>:OUTPUT: Received shuttle (source, client_id, operation_id, self_id)  r1 c1 2 r2
[2017-11-20 23:37:00,150] replica.Replica<Replica:5c004>:OUTPUT: Received result shuttle (source, client_id, operation_id, self_id)  r2 c1 1 r1
[2017-11-20 23:37:00,150] replica.Replica<Replica:5c005>:OUTPUT: Checking for failure associated with trigger, application:  ('shuttle', 'pre')  in replica:  r2
[2017-11-20 23:37:00,150] replica.Replica<Replica:5c004>:OUTPUT: Checking for failure associated with trigger, application:  ('result_shuttle', 'pre')  in replica:  r1
[2017-11-20 23:37:00,151] replica.Replica<Replica:5c005>:OUTPUT: No failure associated with trigger:  shuttle  found in replica:  r2
[2017-11-20 23:37:00,152] replica.Replica<Replica:5c004>:OUTPUT: No failure associated with trigger:  result_shuttle  found in replica:  r1
[2017-11-20 23:37:00,175] replica.Replica<Replica:5c005>:OUTPUT: Checking for failure associated with trigger, application:  ('shuttle', 'post')  in replica:  r2
[2017-11-20 23:37:00,176] replica.Replica<Replica:5c005>:OUTPUT: No failure associated with trigger:  shuttle  found in replica:  r2
[2017-11-20 23:37:00,180] replica.Replica<Replica:5c006>:OUTPUT: Received shuttle (source, client_id, operation_id, self_id)  r2 c1 2 r3
[2017-11-20 23:37:00,181] replica.Replica<Replica:5c006>:OUTPUT: Checking for failure associated with trigger, application:  ('shuttle', 'pre')  in replica:  r3
[2017-11-20 23:37:00,181] replica.Replica<Replica:5c004>:OUTPUT: Checking for failure associated with trigger, application:  ('result_shuttle', 'post')  in replica:  r1
[2017-11-20 23:37:00,181] replica.Replica<Replica:5c006>:OUTPUT: No failure associated with trigger:  shuttle  found in replica:  r3
[2017-11-20 23:37:00,182] replica.Replica<Replica:5c004>:OUTPUT: No failure associated with trigger:  result_shuttle  found in replica:  r1
[2017-11-20 23:37:00,188] replica.Replica<Replica:5c003>:OUTPUT: Received result shuttle (source, client_id, operation_id, self_id)  r1 c1 1 r0
[2017-11-20 23:37:00,189] replica.Replica<Replica:5c003>:OUTPUT: Checking for failure associated with trigger, application:  ('result_shuttle', 'pre')  in replica:  r0
[2017-11-20 23:37:00,189] replica.Replica<Replica:5c003>:OUTPUT: No failure associated with trigger:  result_shuttle  found in replica:  r0
[2017-11-20 23:37:00,192] replica.Replica<Replica:5c007>:OUTPUT: Checking for failure associated with trigger, application:  ('shuttle', 'post')  in replica:  r4
[2017-11-20 23:37:00,192] replica.Replica<Replica:5c007>:OUTPUT: No failure associated with trigger:  shuttle  found in replica:  r4
[2017-11-20 23:37:00,198] replica.Replica<Replica:5c007>:OUTPUT: Checking for failure associated with trigger, application:  ('shuttle', 'post')  in replica:  r4
[2017-11-20 23:37:00,199] replica.Replica<Replica:5c007>:OUTPUT: No failure associated with trigger:  shuttle  found in replica:  r4
[2017-11-20 23:37:00,203] client.Client<Client:5c001>:OUTPUT: A result was received from replica= r4
[2017-11-20 23:37:00,207] replica.Replica<Replica:5c006>:OUTPUT: Checking for failure associated with trigger, application:  ('shuttle', 'post')  in replica:  r3
[2017-11-20 23:37:00,207] replica.Replica<Replica:5c006>:OUTPUT: No failure associated with trigger:  shuttle  found in replica:  r3
[2017-11-20 23:37:00,212] replica.Replica<Replica:5c007>:OUTPUT: Received shuttle (source, client_id, operation_id, self_id)  r3 c1 2 r4
 2 r3
[2017-11-20 23:37:00,212] replica.Replica<Replica:5c007>:OUTPUT: Checking for failure associated with trigger, application:  ('shuttle', 'pre')  in replica:  r4
[2017-11-20 23:37:00,213] replica.Replica<Replica:5c007>:OUTPUT: No failure associated with trigger:  shuttle  found in replica:  r4
[2017-11-20 23:37:00,212] replica.Replica<Replica:5c006>:OUTPUT: Checking for failure associated with trigger, application:  ('result_shuttle', 'pre')  in replica:  r3
[2017-11-20 23:37:00,217] replica.Replica<Replica:5c006>:OUTPUT: No failure associated with trigger:  result_shuttle  found in replica:  r3
[2017-11-20 23:37:00,226] client.Client<Client:5c001>:OUTPUT: The operation result pair is= {'id': 2, 'client_id': 'c0', 'command_string': "append('movie', '123')"} OK
[2017-11-20 23:37:00,227] client.Client<Client:5c001>:OUTPUT: The result received is valid:(result, operation)= OK {'id': 2, 'client_id': 'c0', 'command_string': "append('movie', '123')"}
[2017-11-20 23:37:00,227] client.Client<Client:5c001>:OUTPUT: The following operation was created: {'id': 3, 'client_id': 'c0', 'command_string': "get('movie')"}
[2017-11-20 23:37:00,227] client.Client<Client:5c001>:OUTPUT: return_value= star123
[2017-11-20 23:37:00,230] client.Client<Client:5c001>:OUTPUT: A request for an operation was made {'content': {'is_retry': False, 'operation': {'id': 3, 'client_id': 'c0', 'command_string': "get('movie')"}}, 'signed_content': b"8{\xb2?\x04\xca\xd0'\xfbeb\x05]\xdf\xe3\xd6\xee\x96\x85\xa4\x9b\xaa`{\xe7\xa49\x8b\xfd\xa0\xc5TC\xdd\xfa\xc9\xa5\x02};\x92;<\x94=\x1d\x91\x1b\r\x82(\xb7\x80=i\x84\xa9A`\xd4`h\xae\x0f\x80\x03}q\x00(X\x08\x00\x00\x00is_retryq\x01\x89X\t\x00\x00\x00operationq\x02}q\x03(X\x02\x00\x00\x00idq\x04K\x03X\t\x00\x00\x00client_idq\x05X\x02\x00\x00\x00c0q\x06X\x0e\x00\x00\x00command_stringq\x07X\x0c\x00\x00\x00get('movie')q\x08uu."}
[2017-11-20 23:37:00,232] replica.Replica<Replica:5c003>:OUTPUT: Received request message (source, client_id, operation_id, self_id)  c0 c0 3 r0
[2017-11-20 23:37:00,232] replica.Replica<Replica:5c003>:OUTPUT: Checking for failure associated with trigger, application:  ('client_request', 'pre')  in replica:  r0
[2017-11-20 23:37:00,233] replica.Replica<Replica:5c003>:OUTPUT: No failure associated with trigger:  client_request  found in replica:  r0
[2017-11-20 23:37:00,238] replica.Replica<Replica:5c003>:OUTPUT: Head is processing request (operation_id, client_id) (3, 'c0')
[2017-11-20 23:37:00,246] replica.Replica<Replica:5c003>:OUTPUT: Checking for failure associated with trigger, application:  ('client_request', 'post')  in replica:  r0
[2017-11-20 23:37:00,247] replica.Replica<Replica:5c006>:OUTPUT: Checking for failure associated with trigger, application:  ('result_shuttle', 'post')  in replica:  r3
[2017-11-20 23:37:00,247] replica.Replica<Replica:5c003>:OUTPUT: No failure associated with trigger:  client_request  found in replica:  r0
[2017-11-20 23:37:00,247] replica.Replica<Replica:5c006>:OUTPUT: No failure associated with trigger:  result_shuttle  found in replica:  r3
[2017-11-20 23:37:00,250] replica.Replica<Replica:5c004>:OUTPUT: Received shuttle (source, client_id, operation_id, self_id)  r0 c0 3 r1
[2017-11-20 23:37:00,250] replica.Replica<Replica:5c004>:OUTPUT: Checking for failure associated with trigger, application:  ('shuttle', 'pre')  in replica:  r1
[2017-11-20 23:37:00,251] replica.Replica<Replica:5c004>:OUTPUT: No failure associated with trigger:  shuttle  found in replica:  r1
[2017-11-20 23:37:00,252] replica.Replica<Replica:5c005>:OUTPUT: Received result shuttle (source, client_id, operation_id, self_id)  r3 c0 2 r2
[2017-11-20 23:37:00,252] replica.Replica<Replica:5c005>:OUTPUT: Checking for failure associated with trigger, application:  ('result_shuttle', 'pre')  in replica:  r2
[2017-11-20 23:37:00,253] replica.Replica<Replica:5c005>:OUTPUT: No failure associated with trigger:  result_shuttle  found in replica:  r2
[2017-11-20 23:37:00,266] replica.Replica<Replica:5c007>:OUTPUT: Checking for failure associated with trigger, application:  ('shuttle', 'post')  in replica:  r4
[2017-11-20 23:37:00,266] replica.Replica<Replica:5c007>:OUTPUT: No failure associated with trigger:  shuttle  found in replica:  r4
[2017-11-20 23:37:00,268] replica.Replica<Replica:5c004>:OUTPUT: Checking for failure associated with trigger, application:  ('shuttle', 'post')  in replica:  r1
[2017-11-20 23:37:00,268] replica.Replica<Replica:5c004>:OUTPUT: No failure associated with trigger:  shuttle  found in replica:  r1
[2017-11-20 23:37:00,271] replica.Replica<Replica:5c006>:OUTPUT: Received result shuttle (source, client_id, operation_id, self_id)  r4 c1 2 r3
[2017-11-20 23:37:00,272] replica.Replica<Replica:5c006>:OUTPUT: Checking for failure associated with trigger, application:  ('result_shuttle', 'pre')  in replica:  r3
[2017-11-20 23:37:00,272] replica.Replica<Replica:5c006>:OUTPUT: No failure associated with trigger:  result_shuttle  found in replica:  r3
[2017-11-20 23:37:00,273] replica.Replica<Replica:5c007>:OUTPUT: Checking for failure associated with trigger, application:  ('shuttle', 'post')  in replica:  r4
[2017-11-20 23:37:00,274] replica.Replica<Replica:5c007>:OUTPUT: No failure associated with trigger:  shuttle  found in replica:  r4
[2017-11-20 23:37:00,278] replica.Replica<Replica:5c005>:OUTPUT: Checking for failure associated with trigger, application:  ('result_shuttle', 'post')  in replica:  r2
[2017-11-20 23:37:00,278] replica.Replica<Replica:5c005>:OUTPUT: No failure associated with trigger:  result_shuttle  found in replica:  r2
[2017-11-20 23:37:00,278] client.Client<Client:5c002>:OUTPUT: A result was received from replica= r4
[2017-11-20 23:37:00,282] replica.Replica<Replica:5c005>:OUTPUT: Received shuttle (source, client_id, operation_id, self_id)  r1 c0 3 r2
[2017-11-20 23:37:00,284] replica.Replica<Replica:5c004>:OUTPUT: Received result shuttle (source, client_id, operation_id, self_id)  r2 c0 2 r1
[2017-11-20 23:37:00,284] replica.Replica<Replica:5c005>:OUTPUT: Checking for failure associated with trigger, application:  ('shuttle', 'pre')  in replica:  r2
[2017-11-20 23:37:00,284] replica.Replica<Replica:5c004>:OUTPUT: Checking for failure associated with trigger, application:  ('result_shuttle', 'pre')  in replica:  r1
[2017-11-20 23:37:00,285] replica.Replica<Replica:5c004>:OUTPUT: No failure associated with trigger:  result_shuttle  found in replica:  r1
[2017-11-20 23:37:00,285] replica.Replica<Replica:5c005>:OUTPUT: No failure associated with trigger:  shuttle  found in replica:  r2
[2017-11-20 23:37:00,301] client.Client<Client:5c002>:OUTPUT: The operation result pair is= {'id': 2, 'client_id': 'c1', 'command_string': "append('movie1', '123')"} OK
[2017-11-20 23:37:00,301] client.Client<Client:5c002>:OUTPUT: The result received is valid:(result, operation)= OK {'id': 2, 'client_id': 'c1', 'command_string': "append('movie1', '123')"}
[2017-11-20 23:37:00,302] client.Client<Client:5c002>:OUTPUT: The following operation was created: {'id': 3, 'client_id': 'c1', 'command_string': "get('movie1')"}
[2017-11-20 23:37:00,303] client.Client<Client:5c002>:OUTPUT: return_value= star1123
[2017-11-20 23:37:00,302] replica.Replica<Replica:5c006>:OUTPUT: Checking for failure associated with trigger, application:  ('result_shuttle', 'post')  in replica:  r3
[2017-11-20 23:37:00,304] replica.Replica<Replica:5c006>:OUTPUT: No failure associated with trigger:  result_shuttle  found in replica:  r3
[2017-11-20 23:37:00,306] client.Client<Client:5c002>:OUTPUT: A request for an operation was made {'content': {'is_retry': False, 'operation': {'id': 3, 'client_id': 'c1', 'command_string': "get('movie1')"}}, 'signed_content': b"\xae\x83\xf0\x03\x08\xa9\xa9:\xfa\xe6`/\xed\x16\xa8\xc7\xaa<Q\xf2~\x81\xe4\xb40\xf1\xc5\x07\x05\xdb\xe5\xd6F{\xfb\x93\x85\xf2\xa2n\xb9\xd6\xcf2\x11\xaey4\x9c#\x8e\xd2C\x96\x07\x0c\xf9\xd8\xf7\x9b{\xdcX\x04\x80\x03}q\x00(X\x08\x00\x00\x00is_retryq\x01\x89X\t\x00\x00\x00operationq\x02}q\x03(X\x02\x00\x00\x00idq\x04K\x03X\t\x00\x00\x00client_idq\x05X\x02\x00\x00\x00c1q\x06X\x0e\x00\x00\x00command_stringq\x07X\r\x00\x00\x00get('movie1')q\x08uu."}
[2017-11-20 23:37:00,307] replica.Replica<Replica:5c003>:OUTPUT: Received request message (source, client_id, operation_id, self_id)  c1 c1 3 r0
[2017-11-20 23:37:00,308] replica.Replica<Replica:5c003>:OUTPUT: Checking for failure associated with trigger, application:  ('client_request', 'pre')  in replica:  r0
[2017-11-20 23:37:00,308] replica.Replica<Replica:5c003>:OUTPUT: No failure associated with trigger:  client_request  found in replica:  r0
[2017-11-20 23:37:00,310] replica.Replica<Replica:5c005>:OUTPUT: Checking for failure associated with trigger, application:  ('shuttle', 'post')  in replica:  r2
[2017-11-20 23:37:00,311] replica.Replica<Replica:5c005>:OUTPUT: No failure associated with trigger:  shuttle  found in replica:  r2
[2017-11-20 23:37:00,311] replica.Replica<Replica:5c003>:OUTPUT: Head is processing request (operation_id, client_id) (3, 'c1')
[2017-11-20 23:37:00,313] replica.Replica<Replica:5c004>:OUTPUT: Checking for failure associated with trigger, application:  ('result_shuttle', 'post')  in replica:  r1
[2017-11-20 23:37:00,315] replica.Replica<Replica:5c006>:OUTPUT: Received shuttle (source, client_id, operation_id, self_id)  r2 c0 3 r3
[2017-11-20 23:37:00,313] replica.Replica<Replica:5c004>:OUTPUT: No failure associated with trigger:  result_shuttle  found in replica:  r1
[2017-11-20 23:37:00,315] replica.Replica<Replica:5c006>:OUTPUT: Checking for failure associated with trigger, application:  ('shuttle', 'pre')  in replica:  r3
[2017-11-20 23:37:00,316] replica.Replica<Replica:5c006>:OUTPUT: No failure associated with trigger:  shuttle  found in replica:  r3
[2017-11-20 23:37:00,318] replica.Replica<Replica:5c005>:OUTPUT: Received result shuttle (source, client_id, operation_id, self_id)  r3 c1 2 r2
post')  in replica:  r0
[2017-11-20 23:37:00,319] replica.Replica<Replica:5c003>:OUTPUT: No failure associated with trigger:  client_request  found in replica:  r0
[2017-11-20 23:37:00,320] replica.Replica<Replica:5c005>:OUTPUT: Checking for failure associated with trigger, application:  ('result_shuttle', 'pre')  in replica:  r2
[2017-11-20 23:37:00,320] replica.Replica<Replica:5c005>:OUTPUT: No failure associated with trigger:  result_shuttle  found in replica:  r2
[2017-11-20 23:37:00,323] replica.Replica<Replica:5c003>:OUTPUT: Received result shuttle (source, client_id, operation_id, self_id)  r1 c0 2 r0
[2017-11-20 23:37:00,324] replica.Replica<Replica:5c004>:OUTPUT: Checking for failure associated with trigger, application:  ('shuttle', 'pre')  in replica:  r1
[2017-11-20 23:37:00,324] replica.Replica<Replica:5c003>:OUTPUT: Checking for failure associated with trigger, application:  ('result_shuttle', 'pre')  in replica:  r0
[2017-11-20 23:37:00,324] replica.Replica<Replica:5c004>:OUTPUT: No failure associated with trigger:  shuttle  found in replica:  r1
[2017-11-20 23:37:00,324] replica.Replica<Replica:5c003>:OUTPUT: No failure associated with trigger:  result_shuttle  found in replica:  r0
[2017-11-20 23:37:00,344] replica.Replica<Replica:5c004>:OUTPUT: Checking for failure associated with trigger, application:  ('shuttle', 'post')  in replica:  r1
[2017-11-20 23:37:00,345] replica.Replica<Replica:5c006>:OUTPUT: Checking for failure associated with trigger, application:  ('shuttle', 'post')  in replica:  r3
[2017-11-20 23:37:00,345] replica.Replica<Replica:5c004>:OUTPUT: No failure associated with trigger:  shuttle  found in replica:  r1
[2017-11-20 23:37:00,345] replica.Replica<Replica:5c006>:OUTPUT: No failure associated with trigger:  shuttle  found in replica:  r3
[2017-11-20 23:37:00,349] replica.Replica<Replica:5c005>:OUTPUT: Checking for failure associated with trigger, application:  ('result_shuttle', 'post')  in replica:  r2
[2017-11-20 23:37:00,350] replica.Replica<Replica:5c005>:OUTPUT: No failure associated with trigger:  result_shuttle  found in replica:  r2
[2017-11-20 23:37:00,350] replica.Replica<Replica:5c007>:OUTPUT: Received shuttle (source, client_id, operation_id, self_id)  r3 c0 3 r4
[2017-11-20 23:37:00,350] replica.Replica<Replica:5c007>:OUTPUT: Checking for failure associated with trigger, application:  ('shuttle', 'pre')  in replica:  r4
[2017-11-20 23:37:00,351] replica.Replica<Replica:5c007>:OUTPUT: No failure associated with trigger:  shuttle  found in replica:  r4
[2017-11-20 23:37:00,355] replica.Replica<Replica:5c005>:OUTPUT: Received shuttle (source, client_id, operation_id, self_id)  r1 c1 3 r2
[2017-11-20 23:37:00,357] replica.Replica<Replica:5c004>:OUTPUT: Received result shuttle (source, client_id, operation_id, self_id)  r2 c1 2 r1
[2017-11-20 23:37:00,359] replica.Replica<Replica:5c005>:OUTPUT: Checking for failure associated with trigger, application:  ('shuttle', 'pre')  in replica:  r2
[2017-11-20 23:37:00,359] replica.Replica<Replica:5c004>:OUTPUT: Checking for failure associated with trigger, application:  ('result_shuttle', 'pre')  in replica:  r1
[2017-11-20 23:37:00,360] replica.Replica<Replica:5c005>:OUTPUT: No failure associated with trigger:  shuttle  found in replica:  r2
[2017-11-20 23:37:00,360] replica.Replica<Replica:5c004>:OUTPUT: No failure associated with trigger:  result_shuttle  found in replica:  r1
[2017-11-20 23:37:00,386] replica.Replica<Replica:5c005>:OUTPUT: Checking for failure associated with trigger, application:  ('shuttle', 'post')  in replica:  r2
[2017-11-20 23:37:00,388] replica.Replica<Replica:5c005>:OUTPUT: No failure associated with trigger:  shuttle  found in replica:  r2
[2017-11-20 23:37:00,392] replica.Replica<Replica:5c004>:OUTPUT: Checking for failure associated with trigger, application:  ('result_shuttle', 'post')  in replica:  r1
[2017-11-20 23:37:00,393] replica.Replica<Replica:5c006>:OUTPUT: Received shuttle (source, client_id, operation_id, self_id)  r2 c1 3 r3
[2017-11-20 23:37:00,393] replica.Replica<Replica:5c004>:OUTPUT: No failure associated with trigger:  result_shuttle  found in replica:  r1
[2017-11-20 23:37:00,393] replica.Replica<Replica:5c006>:OUTPUT: Checking for failure associated with trigger, application:  ('shuttle', 'pre')  in replica:  r3
[2017-11-20 23:37:00,393] replica.Replica<Replica:5c006>:OUTPUT: No failure associated with trigger:  shuttle  found in replica:  r3
[2017-11-20 23:37:00,398] replica.Replica<Replica:5c003>:OUTPUT: Received result shuttle (source, client_id, operation_id, self_id)  r1 c1 2 r0
[2017-11-20 23:37:00,399] replica.Replica<Replica:5c003>:OUTPUT: Checking for failure associated with trigger, application:  ('result_shuttle', 'pre')  in replica:  r0
[2017-11-20 23:37:00,399] replica.Replica<Replica:5c003>:OUTPUT: No failure associated with trigger:  result_shuttle  found in replica:  r0
[2017-11-20 23:37:00,405] replica.Replica<Replica:5c007>:OUTPUT: Checking for failure associated with trigger, application:  ('shuttle', 'post')  in replica:  r4
[2017-11-20 23:37:00,405] replica.Replica<Replica:5c007>:OUTPUT: No failure associated with trigger:  shuttle  found in replica:  r4
[2017-11-20 23:37:00,411] replica.Replica<Replica:5c007>:OUTPUT: Checking for failure associated with trigger, application:  ('shuttle', 'post')  in replica:  r4
[2017-11-20 23:37:00,412] replica.Replica<Replica:5c007>:OUTPUT: No failure associated with trigger:  shuttle  found in replica:  r4
[2017-11-20 23:37:00,415] client.Client<Client:5c001>:OUTPUT: A result was received from replica= r4
[2017-11-20 23:37:00,416] replica.Replica<Replica:5c006>:OUTPUT: Checking for failure associated with trigger, application:  ('shuttle', 'post')  in replica:  r3
[2017-11-20 23:37:00,417] replica.Replica<Replica:5c006>:OUTPUT: No failure associated with trigger:  shuttle  found in replica:  r3
[2017-11-20 23:37:00,422] replica.Replica<Replica:5c006>:OUTPUT: Received result shuttle (source, client_id, operation_id, self_id)  r4 c0 3 r3
[2017-11-20 23:37:00,425] replica.Replica<Replica:5c007>:OUTPUT: Received shuttle (source, client_id, operation_id, self_id)  r3 c1 3 r4
[2017-11-20 23:37:00,425] replica.Replica<Replica:5c006>:OUTPUT: Checking for failure associated with trigger, application:  ('result_shuttle', 'pre')  in replica:  r3
[2017-11-20 23:37:00,426] replica.Replica<Replica:5c007>:OUTPUT: Checking for failure associated with trigger, application:  ('shuttle', 'pre')  in replica:  r4
[2017-11-20 23:37:00,426] replica.Replica<Replica:5c007>:OUTPUT: No failure associated with trigger:  shuttle  found in replica:  r4
[2017-11-20 23:37:00,426] replica.Replica<Replica:5c006>:OUTPUT: No failure associated with trigger:  result_shuttle  found in replica:  r3
[2017-11-20 23:37:00,438] client.Client<Client:5c001>:OUTPUT: The operation result pair is= {'id': 3, 'client_id': 'c0', 'command_string': "get('movie')"} star123
[2017-11-20 23:37:00,439] client.Client<Client:5c001>:OUTPUT: The result received is valid:(result, operation)= star123 {'id': 3, 'client_id': 'c0', 'command_string': "get('movie')"}
[2017-11-20 23:37:00,439] client.Client<Client:5c001>:OUTPUT: The following operation was created: {'id': 4, 'client_id': 'c0', 'command_string': "get('movie');"}
[2017-11-20 23:37:00,440] client.Client<Client:5c001>:OUTPUT: return_value= star123
[2017-11-20 23:37:00,443] client.Client<Client:5c001>:OUTPUT: A request for an operation was made {'content': {'is_retry': False, 'operation': {'id': 4, 'client_id': 'c0', 'command_string': "get('movie');"}}, 'signed_content': b"\x94N$\xa4\x02{e+\xd2\xacwVI\xe6\xe2\xd1D\xe2'n\x9b\xee\xf7\xf51\x9a\x01\x97\xce\x17\xdf\x16]V\x82a\x11f\xd2\xbf\xabzK\xe5$t\xf9\x1e\xc7\xb7\x814L 9\x11\\l\xdb\xa1*&\xa8\x0f\x80\x03}q\x00(X\x08\x00\x00\x00is_retryq\x01\x89X\t\x00\x00\x00operationq\x02}q\x03(X\x02\x00\x00\x00idq\x04K\x04X\t\x00\x00\x00client_idq\x05X\x02\x00\x00\x00c0q\x06X\x0e\x00\x00\x00command_stringq\x07X\r\x00\x00\x00get('movie');q\x08uu."}
[2017-11-20 23:37:00,445] replica.Replica<Replica:5c003>:OUTPUT: Received request message (source, client_id, operation_id, self_id)  c0 c0 4 r0
[2017-11-20 23:37:00,445] replica.Replica<Replica:5c003>:OUTPUT: Checking for failure associated with trigger, application:  ('client_request', 'pre')  in replica:  r0
[2017-11-20 23:37:00,445] replica.Replica<Replica:5c003>:OUTPUT: No failure associated with trigger:  client_request  found in replica:  r0
[2017-11-20 23:37:00,449] replica.Replica<Replica:5c003>:OUTPUT: Head is processing request (operation_id, client_id) (4, 'c0')
[2017-11-20 23:37:00,456] replica.Replica<Replica:5c006>:OUTPUT: Checking for failure associated with trigger, application:  ('result_shuttle', 'post')  in replica:  r3
[2017-11-20 23:37:00,456] replica.Replica<Replica:5c003>:OUTPUT: Checking for failure associated with trigger, application:  ('client_request', 'post')  in replica:  r0
[2017-11-20 23:37:00,457] replica.Replica<Replica:5c006>:OUTPUT: No failure associated with trigger:  result_shuttle  found in replica:  r3
[2017-11-20 23:37:00,457] replica.Replica<Replica:5c003>:OUTPUT: No failure associated with trigger:  client_request  found in replica:  r0
[2017-11-20 23:37:00,460] replica.Replica<Replica:5c004>:OUTPUT: Received shuttle (source, client_id, operation_id, self_id)  r0 c0 4 r1
[2017-11-20 23:37:00,461] replica.Replica<Replica:5c004>:OUTPUT: Checking for failure associated with trigger, application:  ('shuttle', 'pre')  in replica:  r1
[2017-11-20 23:37:00,461] replica.Replica<Replica:5c004>:OUTPUT: No failure associated with trigger:  shuttle  found in replica:  r1
[2017-11-20 23:37:00,462] replica.Replica<Replica:5c005>:OUTPUT: Received result shuttle (source, client_id, operation_id, self_id)  r3 c0 3 r2
[2017-11-20 23:37:00,462] replica.Replica<Replica:5c005>:OUTPUT: Checking for failure associated with trigger, application:  ('result_shuttle', 'pre')  in replica:  r2
[2017-11-20 23:37:00,463] replica.Replica<Replica:5c005>:OUTPUT: No failure associated with trigger:  result_shuttle  found in replica:  r2
[2017-11-20 23:37:00,475] replica.Replica<Replica:5c007>:OUTPUT: Checking for failure associated with trigger, application:  ('shuttle', 'post')  in replica:  r4
[2017-11-20 23:37:00,476] replica.Replica<Replica:5c007>:OUTPUT: No failure associated with trigger:  shuttle  found in replica:  r4
[2017-11-20 23:37:00,478] replica.Replica<Replica:5c004>:OUTPUT: Checking for failure associated with trigger, application:  ('shuttle', 'post')  in replica:  r1
[2017-11-20 23:37:00,478] replica.Replica<Replica:5c004>:OUTPUT: No failure associated with trigger:  shuttle  found in replica:  r1
[2017-11-20 23:37:00,481] replica.Replica<Replica:5c006>:OUTPUT: Received result shuttle (source, client_id, operation_id, self_id)  r4 c1 3 r3
[2017-11-20 23:37:00,482] replica.Replica<Replica:5c006>:OUTPUT: Checking for failure associated with trigger, application:  ('result_shuttle', 'pre')  in replica:  r3
[2017-11-20 23:37:00,482] replica.Replica<Replica:5c007>:OUTPUT: Checking for failure associated with trigger, application:  ('shuttle', 'post')  in replica:  r4
[2017-11-20 23:37:00,482] replica.Replica<Replica:5c006>:OUTPUT: No failure associated with trigger:  result_shuttle  found in replica:  r3
[2017-11-20 23:37:00,482] replica.Replica<Replica:5c007>:OUTPUT: No failure associated with trigger:  shuttle  found in replica:  r4
[2017-11-20 23:37:00,485] client.Client<Client:5c002>:OUTPUT: A result was received from replica= r4
[2017-11-20 23:37:00,487] replica.Replica<Replica:5c005>:OUTPUT: Checking for failure associated with trigger, application:  ('result_shuttle', 'post')  in replica:  r2
[2017-11-20 23:37:00,488] replica.Replica<Replica:5c005>:OUTPUT: No failure associated with trigger:  result_shuttle  found in replica:  r2
[2017-11-20 23:37:00,494] replica.Replica<Replica:5c004>:OUTPUT: Received result shuttle (source, client_id, operation_id, self_id)  r2 c0 3 r1
[2017-11-20 23:37:00,495] replica.Replica<Replica:5c004>:OUTPUT: Checking for failure associated with trigger, application:  ('result_shuttle', 'pre')  in replica:  r1
[2017-11-20 23:37:00,493] replica.Replica<Replica:5c005>:OUTPUT: Received shuttle (source, client_id, operation_id, self_id)  r1 c0 4 r2
[2017-11-20 23:37:00,495] replica.Replica<Replica:5c004>:OUTPUT: No failure associated with trigger:  result_shuttle  found in replica:  r1
[2017-11-20 23:37:00,496] replica.Replica<Replica:5c005>:OUTPUT: Checking for failure associated with trigger, application:  ('shuttle', 'pre')  in replica:  r2
[2017-11-20 23:37:00,496] replica.Replica<Replica:5c005>:OUTPUT: No failure associated with trigger:  shuttle  found in replica:  r2
[2017-11-20 23:37:00,508] client.Client<Client:5c002>:OUTPUT: The operation result pair is= {'id': 3, 'client_id': 'c1', 'command_string': "get('movie1')"} star1123
[2017-11-20 23:37:00,509] client.Client<Client:5c002>:OUTPUT: The result received is valid:(result, operation)= star1123 {'id': 3, 'client_id': 'c1', 'command_string': "get('movie1')"}
[2017-11-20 23:37:00,509] client.Client<Client:5c002>:OUTPUT: The following operation was created: {'id': 4, 'client_id': 'c1', 'command_string': "get('movie1');"}
[2017-11-20 23:37:00,509] client.Client<Client:5c002>:OUTPUT: return_value= star1123
[2017-11-20 23:37:00,513] client.Client<Client:5c002>:OUTPUT: A request for an operation was made {'content': {'is_retry': False, 'operation': {'id': 4, 'client_id': 'c1', 'command_string': "get('movie1');"}}, 'signed_content': b"\xebt\xa2\xed\x96\x9bj\xfb\xe3\x96,\xe4\xb8\x10\x16\xd9\xcb\xb0Q:\xc2)\xcf\xa3p_`6\xa9\x8c\x98:o\xf2a\xde\xe8p\x0f \xcb\xce\x02\xf0\x00].\xa2\x03=\x82p$\xd9d\x1d\xd4\x0b&\xe3\x1c\x0f\xd7\n\x80\x03}q\x00(X\x08\x00\x00\x00is_retryq\x01\x89X\t\x00\x00\x00operationq\x02}q\x03(X\x02\x00\x00\x00idq\x04K\x04X\t\x00\x00\x00client_idq\x05X\x02\x00\x00\x00c1q\x06X\x0e\x00\x00\x00command_stringq\x07X\x0e\x00\x00\x00get('movie1');q\x08uu."}
[2017-11-20 23:37:00,514] replica.Replica<Replica:5c003>:OUTPUT: Received request message (source, client_id, operation_id, self_id)  c1 c1 4 r0
[2017-11-20 23:37:00,514] replica.Replica<Replica:5c006>:OUTPUT: Checking for failure associated with trigger, application:  ('result_shuttle', 'post')  in replica:  r3
[2017-11-20 23:37:00,514] replica.Replica<Replica:5c003>:OUTPUT: Checking for failure associated with trigger, application:  ('client_request', 'pre')  in replica:  r0
[2017-11-20 23:37:00,515] replica.Replica<Replica:5c006>:OUTPUT: No failure associated with trigger:  result_shuttle  found in replica:  r3
[2017-11-20 23:37:00,515] replica.Replica<Replica:5c003>:OUTPUT: No failure associated with trigger:  client_request  found in replica:  r0
[2017-11-20 23:37:00,519] replica.Replica<Replica:5c005>:OUTPUT: Checking for failure associated with trigger, application:  ('shuttle', 'post')  in replica:  r2
[2017-11-20 23:37:00,520] replica.Replica<Replica:5c005>:OUTPUT: No failure associated with trigger:  shuttle  found in replica:  r2
[2017-11-20 23:37:00,520] replica.Replica<Replica:5c003>:OUTPUT: Head is processing request (operation_id, client_id) (4, 'c1')
[2017-11-20 23:37:00,522] replica.Replica<Replica:5c004>:OUTPUT: Checking for failure associated with trigger, application:  ('result_shuttle', 'post')  in replica:  r1
[2017-11-20 23:37:00,523] replica.Replica<Replica:5c004>:OUTPUT: No failure associated with trigger:  result_shuttle  found in replica:  r1
[2017-11-20 23:37:00,524] replica.Replica<Replica:5c006>:OUTPUT: Received shuttle (source, client_id, operation_id, self_id)  r2 c0 4 r3
[2017-11-20 23:37:00,524] replica.Replica<Replica:5c005>:OUTPUT: Received result shuttle (source, client_id, operation_id, self_id)  r3 c1 3 r2
[2017-11-20 23:37:00,524] replica.Replica<Replica:5c006>:OUTPUT: Checking for failure associated with trigger, application:  ('shuttle', 'pre')  in replica:  r3
[2017-11-20 23:37:00,525] replica.Replica<Replica:5c005>:OUTPUT: Checking for failure associated with trigger, application:  ('result_shuttle', 'pre')  in replica:  r2
[2017-11-20 23:37:00,526] replica.Replica<Replica:5c006>:OUTPUT: No failure associated with trigger:  shuttle  found in replica:  r3
[2017-11-20 23:37:00,527] replica.Replica<Replica:5c005>:OUTPUT: No failure associated with trigger:  result_shuttle  found in replica:  r2
[2017-11-20 23:37:00,528] replica.Replica<Replica:5c003>:OUTPUT: Checking for failure associated with trigger, application:  ('client_request', 'post')  in replica:  r0
[2017-11-20 23:37:00,529] replica.Replica<Replica:5c003>:OUTPUT: No failure associated with trigger:  client_request  found in replica:  r0
[2017-11-20 23:37:00,532] replica.Replica<Replica:5c004>:OUTPUT: Received shuttle (source, client_id, operation_id, self_id)  r0 c1 4 r1
[2017-11-20 23:37:00,532] replica.Replica<Replica:5c003>:OUTPUT: Received result shuttle (source, client_id, operation_id, self_id)  r1 c0 3 r0
[2017-11-20 23:37:00,533] replica.Replica<Replica:5c004>:OUTPUT: Checking for failure associated with trigger, application:  ('shuttle', 'pre')  in replica:  r1
[2017-11-20 23:37:00,533] replica.Replica<Replica:5c003>:OUTPUT: Checking for failure associated with trigger, application:  ('result_shuttle', 'pre')  in replica:  r0
[2017-11-20 23:37:00,533] replica.Replica<Replica:5c004>:OUTPUT: No failure associated with trigger:  shuttle  found in replica:  r1
[2017-11-20 23:37:00,533] replica.Replica<Replica:5c003>:OUTPUT: No failure associated with trigger:  result_shuttle  found in replica:  r0
[2017-11-20 23:37:00,552] replica.Replica<Replica:5c004>:OUTPUT: Checking for failure associated with trigger, application:  ('shuttle', 'post')  in replica:  r1
[2017-11-20 23:37:00,553] replica.Replica<Replica:5c004>:OUTPUT: No failure associated with trigger:  shuttle  found in replica:  r1
[2017-11-20 23:37:00,557] replica.Replica<Replica:5c005>:OUTPUT: Checking for failure associated with trigger, application:  ('result_shuttle', 'post')  in replica:  r2
[2017-11-20 23:37:00,557] replica.Replica<Replica:5c005>:OUTPUT: No failure associated with trigger:  result_shuttle  found in replica:  r2
[2017-11-20 23:37:00,558] replica.Replica<Replica:5c006>:OUTPUT: Checking for failure associated with trigger, application:  ('shuttle', 'post')  in replica:  r3
[2017-11-20 23:37:00,558] replica.Replica<Replica:5c006>:OUTPUT: No failure associated with trigger:  shuttle  found in replica:  r3
[2017-11-20 23:37:00,561] replica.Replica<Replica:5c005>:OUTPUT: Received shuttle (source, client_id, operation_id, self_id)  r1 c1 4 r2
[2017-11-20 23:37:00,562] replica.Replica<Replica:5c004>:OUTPUT: Received result shuttle (source, client_id, operation_id, self_id)  r2 c1 3 r1
[2017-11-20 23:37:00,562] replica.Replica<Replica:5c005>:OUTPUT: Checking for failure associated with trigger, application:  ('shuttle', 'pre')  in replica:  r2
[2017-11-20 23:37:00,562] replica.Replica<Replica:5c004>:OUTPUT: Checking for failure associated with trigger, application:  ('result_shuttle', 'pre')  in replica:  r1
[2017-11-20 23:37:00,562] replica.Replica<Replica:5c005>:OUTPUT: No failure associated with trigger:  shuttle  found in replica:  r2
[2017-11-20 23:37:00,563] replica.Replica<Replica:5c004>:OUTPUT: No failure associated with trigger:  result_shuttle  found in replica:  r1
[2017-11-20 23:37:00,563] replica.Replica<Replica:5c007>:OUTPUT: Received shuttle (source, client_id, operation_id, self_id)  r3 c0 4 r4
[2017-11-20 23:37:00,563] replica.Replica<Replica:5c007>:OUTPUT: Checking for failure associated with trigger, application:  ('shuttle', 'pre')  in replica:  r4
[2017-11-20 23:37:00,564] replica.Replica<Replica:5c007>:OUTPUT: No failure associated with trigger:  shuttle  found in replica:  r4
[2017-11-20 23:37:00,586] replica.Replica<Replica:5c005>:OUTPUT: Checking for failure associated with trigger, application:  ('shuttle', 'post')  in replica:  r2
[2017-11-20 23:37:00,587] replica.Replica<Replica:5c005>:OUTPUT: No failure associated with trigger:  shuttle  found in replica:  r2
[2017-11-20 23:37:00,589] replica.Replica<Replica:5c004>:OUTPUT: Checking for failure associated with trigger, application:  ('result_shuttle', 'post')  in replica:  r1
[2017-11-20 23:37:00,591] replica.Replica<Replica:5c004>:OUTPUT: No failure associated with trigger:  result_shuttle  found in replica:  r1
[2017-11-20 23:37:00,592] replica.Replica<Replica:5c006>:OUTPUT: Received shuttle (source, client_id, operation_id, self_id)  r2 c1 4 r3
[2017-11-20 23:37:00,593] replica.Replica<Replica:5c006>:OUTPUT: Checking for failure associated with trigger, application:  ('shuttle', 'pre')  in replica:  r3
[2017-11-20 23:37:00,593] replica.Replica<Replica:5c006>:OUTPUT: No failure associated with trigger:  shuttle  found in replica:  r3
[2017-11-20 23:37:00,597] replica.Replica<Replica:5c003>:OUTPUT: Received result shuttle (source, client_id, operation_id, self_id)  r1 c1 3 r0
[2017-11-20 23:37:00,597] replica.Replica<Replica:5c003>:OUTPUT: Checking for failure associated with trigger, application:  ('result_shuttle', 'pre')  in replica:  r0
[2017-11-20 23:37:00,598] replica.Replica<Replica:5c003>:OUTPUT: No failure associated with trigger:  result_shuttle  found in replica:  r0
[2017-11-20 23:37:00,608] replica.Replica<Replica:5c007>:OUTPUT: Checking for failure associated with trigger, application:  ('shuttle', 'post')  in replica:  r4
[2017-11-20 23:37:00,609] replica.Replica<Replica:5c007>:OUTPUT: No failure associated with trigger:  shuttle  found in replica:  r4
[2017-11-20 23:37:00,616] replica.Replica<Replica:5c007>:OUTPUT: Checking for failure associated with trigger, application:  ('shuttle', 'post')  in replica:  r4
[2017-11-20 23:37:00,617] replica.Replica<Replica:5c006>:OUTPUT: Checking for failure associated with trigger, application:  ('shuttle', 'post')  in replica:  r3
[2017-11-20 23:37:00,617] replica.Replica<Replica:5c007>:OUTPUT: No failure associated with trigger:  shuttle  found in replica:  r4
[2017-11-20 23:37:00,617] replica.Replica<Replica:5c006>:OUTPUT: No failure associated with trigger:  shuttle  found in replica:  r3
[2017-11-20 23:37:00,620] client.Client<Client:5c001>:OUTPUT: A result was received from replica= r4
[2017-11-20 23:37:00,622] replica.Replica<Replica:5c007>:OUTPUT: Received shuttle (source, client_id, operation_id, self_id)  r3 c1 4 r4
[2017-11-20 23:37:00,622] replica.Replica<Replica:5c006>:OUTPUT: Received result shuttle (source, client_id, operation_id, self_id)  r4 c0 4 r3
[2017-11-20 23:37:00,623] replica.Replica<Replica:5c007>:OUTPUT: Checking for failure associated with trigger, application:  ('shuttle', 'pre')  in replica:  r4
[2017-11-20 23:37:00,624] replica.Replica<Replica:5c006>:OUTPUT: Checking for failure associated with trigger, application:  ('result_shuttle', 'pre')  in replica:  r3
[2017-11-20 23:37:00,624] replica.Replica<Replica:5c007>:OUTPUT: No failure associated with trigger:  shuttle  found in replica:  r4
[2017-11-20 23:37:00,624] replica.Replica<Replica:5c006>:OUTPUT: No failure associated with trigger:  result_shuttle  found in replica:  r3
[2017-11-20 23:37:00,643] client.Client<Client:5c001>:OUTPUT: The operation result pair is= {'id': 4, 'client_id': 'c0', 'command_string': "get('movie');"} star123
[2017-11-20 23:37:00,644] client.Client<Client:5c001>:OUTPUT: The result received is valid:(result, operation)= star123 {'id': 4, 'client_id': 'c0', 'command_string': "get('movie');"}
[2017-11-20 23:37:00,644] client.Client<Client:5c001>:OUTPUT: ~~~~~~~~~~Final state: {'movie': 'star123'}
[2017-11-20 23:37:00,644] client.Client<Client:5c001>:OUTPUT: Requesting shared object from head from client: c0
[2017-11-20 23:37:00,648] replica.Replica<Replica:5c006>:OUTPUT: Checking for failure associated with trigger, application:  ('result_shuttle', 'post')  in replica:  r3
[2017-11-20 23:37:00,649] replica.Replica<Replica:5c006>:OUTPUT: No failure associated with trigger:  result_shuttle  found in replica:  r3
[2017-11-20 23:37:00,649] replica.Replica<Replica:5c003>:OUTPUT: Received wedge request from Olympus in Replica:  r0
[2017-11-20 23:37:00,653] replica.Replica<Replica:5c003>:OUTPUT: Received shared_object_request from client= c0
[2017-11-20 23:37:00,655] replica.Replica<Replica:5c005>:OUTPUT: Received result shuttle (source, client_id, operation_id, self_id)  r3 c0 4 r2
[2017-11-20 23:37:00,655] replica.Replica<Replica:5c005>:OUTPUT: Checking for failure associated with trigger, application:  ('result_shuttle', 'pre')  in replica:  r2
[2017-11-20 23:37:00,656] replica.Replica<Replica:5c005>:OUTPUT: No failure associated with trigger:  result_shuttle  found in replica:  r2
[2017-11-20 23:37:00,659] client.Client<Client:5c001>:OUTPUT: Received shared object from head =  {'movie': 'star123', 'movie1': 'star1123'}
[2017-11-20 23:37:00,663] client.Client<Client:5c001>:OUTPUT: Final results are consistent
[2017-11-20 23:37:00,664] client.Client<Client:5c001>:OUTPUT: Workload finished for client:  c0
[2017-11-20 23:37:00,664] client.Client<Client:5c001>:OUTPUT: Sending done message to Olympus and terminating
[2017-11-20 23:37:00,669] olympus.Olympus<Olympus:bb002>:OUTPUT: Done message received from client:  c0
[2017-11-20 23:37:00,677] replica.Replica<Replica:5c007>:OUTPUT: Checking for failure associated with trigger, application:  ('shuttle', 'post')  in replica:  r4
[2017-11-20 23:37:00,678] replica.Replica<Replica:5c007>:OUTPUT: No failure associated with trigger:  shuttle  found in replica:  r4
[2017-11-20 23:37:00,680] replica.Replica<Replica:5c005>:OUTPUT: Checking for failure associated with trigger, application:  ('result_shuttle', 'post')  in replica:  r2
[2017-11-20 23:37:00,680] replica.Replica<Replica:5c005>:OUTPUT: No failure associated with trigger:  result_shuttle  found in replica:  r2
[2017-11-20 23:37:00,684] replica.Replica<Replica:5c006>:OUTPUT: Received result shuttle (source, client_id, operation_id, self_id)  r4 c1 4 r3
[2017-11-20 23:37:00,685] replica.Replica<Replica:5c006>:OUTPUT: Checking for failure associated with trigger, application:  ('result_shuttle', 'pre')  in replica:  r3
[2017-11-20 23:37:00,685] replica.Replica<Replica:5c006>:OUTPUT: No failure associated with trigger:  result_shuttle  found in replica:  r3
[2017-11-20 23:37:00,688] replica.Replica<Replica:5c004>:OUTPUT: Received result shuttle (source, client_id, operation_id, self_id)  r2 c0 4 r1
[2017-11-20 23:37:00,690] replica.Replica<Replica:5c004>:OUTPUT: Checking for failure associated with trigger, application:  ('result_shuttle', 'pre')  in replica:  r1
[2017-11-20 23:37:00,690] replica.Replica<Replica:5c004>:OUTPUT: No failure associated with trigger:  result_shuttle  found in replica:  r1
[2017-11-20 23:37:00,685] replica.Replica<Replica:5c007>:OUTPUT: Checking for failure associated with trigger, application:  ('shuttle', 'post')  in replica:  r4
[2017-11-20 23:37:00,694] replica.Replica<Replica:5c007>:OUTPUT: No failure associated with trigger:  shuttle  found in replica:  r4
[2017-11-20 23:37:00,698] client.Client<Client:5c002>:OUTPUT: A result was received from replica= r4
[2017-11-20 23:37:00,710] replica.Replica<Replica:5c006>:OUTPUT: Checking for failure associated with trigger, application:  ('result_shuttle', 'post')  in replica:  r3
[2017-11-20 23:37:00,711] replica.Replica<Replica:5c006>:OUTPUT: No failure associated with trigger:  result_shuttle  found in replica:  r3
[2017-11-20 23:37:00,716] replica.Replica<Replica:5c004>:OUTPUT: Checking for failure associated with trigger, application:  ('result_shuttle', 'post')  in replica:  r1
[2017-11-20 23:37:00,716] replica.Replica<Replica:5c005>:OUTPUT: Received result shuttle (source, client_id, operation_id, self_id)  r3 c1 4 r2
[2017-11-20 23:37:00,716] replica.Replica<Replica:5c004>:OUTPUT: No failure associated with trigger:  result_shuttle  found in replica:  r1
[2017-11-20 23:37:00,717] replica.Replica<Replica:5c005>:OUTPUT: Checking for failure associated with trigger, application:  ('result_shuttle', 'pre')  in replica:  r2
[2017-11-20 23:37:00,717] replica.Replica<Replica:5c005>:OUTPUT: No failure associated with trigger:  result_shuttle  found in replica:  r2
[2017-11-20 23:37:00,720] client.Client<Client:5c002>:OUTPUT: The operation result pair is= {'id': 4, 'client_id': 'c1', 'command_string': "get('movie1');"} star1123
[2017-11-20 23:37:00,720] client.Client<Client:5c002>:OUTPUT: The result received is valid:(result, operation)= star1123 {'id': 4, 'client_id': 'c1', 'command_string': "get('movie1');"}
[2017-11-20 23:37:00,721] client.Client<Client:5c002>:OUTPUT: ~~~~~~~~~~Final state: {'movie1': 'star1123'}
[2017-11-20 23:37:00,721] client.Client<Client:5c002>:OUTPUT: Requesting shared object from head from client: c1
[2017-11-20 23:37:00,722] replica.Replica<Replica:5c003>:OUTPUT: Received result shuttle (source, client_id, operation_id, self_id)  r1 c0 4 r0
[2017-11-20 23:37:00,722] replica.Replica<Replica:5c003>:OUTPUT: Checking for failure associated with trigger, application:  ('result_shuttle', 'pre')  in replica:  r0
[2017-11-20 23:37:00,723] replica.Replica<Replica:5c003>:OUTPUT: No failure associated with trigger:  result_shuttle  found in replica:  r0
[2017-11-20 23:37:00,739] replica.Replica<Replica:5c005>:OUTPUT: Checking for failure associated with trigger, application:  ('result_shuttle', 'post')  in replica:  r2
[2017-11-20 23:37:00,740] replica.Replica<Replica:5c005>:OUTPUT: No failure associated with trigger:  result_shuttle  found in replica:  r2
[2017-11-20 23:37:00,745] replica.Replica<Replica:5c004>:OUTPUT: Received result shuttle (source, client_id, operation_id, self_id)  r2 c1 4 r1
[2017-11-20 23:37:00,746] replica.Replica<Replica:5c004>:OUTPUT: Checking for failure associated with trigger, application:  ('result_shuttle', 'pre')  in replica:  r1
[2017-11-20 23:37:00,747] replica.Replica<Replica:5c004>:OUTPUT: No failure associated with trigger:  result_shuttle  found in replica:  r1
[2017-11-20 23:37:00,747] replica.Replica<Replica:5c003>:OUTPUT: Received wedge request from Olympus in Replica:  r0
[2017-11-20 23:37:00,751] replica.Replica<Replica:5c003>:OUTPUT: Received shared_object_request from client= c1
[2017-11-20 23:37:00,758] client.Client<Client:5c002>:OUTPUT: Received shared object from head =  {'movie': 'star123', 'movie1': 'star1123'}
[2017-11-20 23:37:00,762] client.Client<Client:5c002>:OUTPUT: Final results are consistent
[2017-11-20 23:37:00,762] client.Client<Client:5c002>:OUTPUT: Workload finished for client:  c1
[2017-11-20 23:37:00,763] client.Client<Client:5c002>:OUTPUT: Sending done message to Olympus and terminating
[2017-11-20 23:37:00,766] olympus.Olympus<Olympus:bb002>:OUTPUT: Done message received from client:  c1
[2017-11-20 23:37:00,770] olympus.Olympus<Olympus:bb002>:OUTPUT: Terminating replicas..
[2017-11-20 23:37:00,774] olympus.Olympus<Olympus:bb002>:OUTPUT: Sent abort command to replica : r0
[2017-11-20 23:37:00,775] replica.Replica<Replica:5c003>:OUTPUT: Received abort message from Olympus in replica:  r0
[2017-11-20 23:37:00,778] olympus.Olympus<Olympus:bb002>:OUTPUT: Sent abort command to replica : r1
[2017-11-20 23:37:00,779] replica.Replica<Replica:5c003>:OUTPUT: Sending aborted message to Olympus from:  r0
[2017-11-20 23:37:00,783] replica.Replica<Replica:5c003>:OUTPUT: Shutting down replica:  r0
[2017-11-20 23:37:00,783] olympus.Olympus<Olympus:bb002>:OUTPUT: Sent abort command to replica : r2
[2017-11-20 23:37:00,784] replica.Replica<Replica:5c005>:OUTPUT: Received abort message from Olympus in replica:  r2
[2017-11-20 23:37:00,787] olympus.Olympus<Olympus:bb002>:OUTPUT: Sent abort command to replica : r3
[2017-11-20 23:37:00,788] replica.Replica<Replica:5c006>:OUTPUT: Received abort message from Olympus in replica:  r3
[2017-11-20 23:37:00,788] replica.Replica<Replica:5c005>:OUTPUT: Sending aborted message to Olympus from:  r2
[2017-11-20 23:37:00,791] olympus.Olympus<Olympus:bb002>:OUTPUT: Sent abort command to replica : r4
[2017-11-20 23:37:00,793] replica.Replica<Replica:5c007>:OUTPUT: Received abort message from Olympus in replica:  r4
[2017-11-20 23:37:00,793] replica.Replica<Replica:5c006>:OUTPUT: Sending aborted message to Olympus from:  r3
[2017-11-20 23:37:00,793] olympus.Olympus<Olympus:bb002>:OUTPUT: Olympus terminating
[2017-11-20 23:37:00,794] replica.Replica<Replica:5c005>:OUTPUT: Shutting down replica:  r2
[2017-11-20 23:37:00,795] olympus.Olympus<Olympus:bb002>:OUTPUT: elapsed time (seconds):  2.869652509689331
[2017-11-20 23:37:00,782] replica.Replica<Replica:5c004>:OUTPUT: Checking for failure associated with trigger, application:  ('result_shuttle', 'post')  in replica:  r1
[2017-11-20 23:37:00,797] replica.Replica<Replica:5c007>:OUTPUT: Sending aborted message to Olympus from:  r4
[2017-11-20 23:37:00,798] replica.Replica<Replica:5c004>:OUTPUT: No failure associated with trigger:  result_shuttle  found in replica:  r1
[2017-11-20 23:37:00,802] replica.Replica<Replica:5c007>:OUTPUT: Shutting down replica:  r4
[2017-11-20 23:37:00,802] replica.Replica<Replica:5c004>:OUTPUT: Received abort message from Olympus in replica:  r1
[2017-11-20 23:37:00,806] replica.Replica<Replica:5c004>:OUTPUT: Sending aborted message to Olympus from:  r1
[2017-11-20 23:37:00,813] replica.Replica<Replica:5c004>:OUTPUT: Shutting down replica:  r1
